\documentclass[11pt]{article}
\usepackage{color}
\usepackage{authblk}%allows footnote format for authors
\usepackage[letterpaper, margin=1in]{geometry} %package that allows changes in margins and header/footers

\newcommand{\mbh}[1]{\textcolor{green}{ \emph{\scriptsize  #1}} } %creating command for Matt's comments
\newcommand{\lwang}[1]{\textcolor{red}{ \emph{\scriptsize  #1}} } %creating command for Li's comments

\title{Crop adaptation through gene flow (outline)}




\begin{document}

\maketitle

\section*{Outline}
\subsection*{Outline parts}
Introgression in Crops (General Information)
Crop Examples
    Maize
    Wheat/Rye
    Barley
    Sunflower
    Tomato?
Non-Crop System Examples?

Single Center of Origin:
Wheat - With wild relatives
Maize - With wild relatives
Potato - With wild relatives
(Cassava)

Other:
Barley - With wild relatives
Tomato (unknown Center of Origin) - With wild relatives
Sunflower (unknown Center of Origin)
Rice - With wild relatives (both domesticated monophyletic varieties)
Common Bean - Two regions of domestication, one or more domestication events within each
(Cotton)
(Millet)
(Squash)

\subsection*{Introduction}
 "Whenever sympatric populations of crops and their wild relatives show a similar trait, there will be several possible explanations: (1) retention of an ancestral trait, (2) convergent or parallel evolution, and (3) introgression.  Perhaps even more difficult is establishing the direction of introgression; does gene flow go from crop into wild species, the reverse, or in both directions?" and "For this reason, introgression may be best studied between crops and their more distant wild relatives where the probability of finding taxon-specific genetic markers is increased." \cite{doebley1990molecular}


1\subsection*{Introgression in Crops (General)}

From {stewart2003transgene}
A very good review.
Hybrid zones make for a great model system of study.
Introgression from wild populations into agronomic populations is potentially beneficial for breeding programmes.
It is generally easier to introgress into crops than introgressing into wild populations (stronger selection, overcoming LD).

From {label32}
Another very good review, with more of a molecular emphasis

From {label34}
The difficulties of utilizing wild relatives in breeding programmes effectively.

From {label6} {label7} {label26} {Label27}
Lots of papers focus on gene flow from crop to wild relative, particularly about transgene escape




Order:
the crop and the wild ancestor
    where
    how long ago
history of the study of that relationship
traits that are conferred
current application (realized or hypothetical)



\subsection*{Maize}

Maize is a classical model in crop species to study the issue of adaptive introgresssion from wild relatives, as it still grows in sympatry with many of its wild relatives \cite{hufford2013}.

Maize was domesticated about 10,000 years ago in Mexico \cite{smith1997initial} \cite{hufford2012comparative} in the Balsas River valley \cite {matsuoka2002single} from teosinte.

Mexican farmers have long been aware of maize-teosinte hybrids, and in some places, such as the Nobogame valley, farmers have observed desirable traits passed to their crops through the hybrid plants \cite{wilkes1977hybridization} \cite{lumholtz1902unknown} \cite{wilkes1970teosinte}.

In particular, \emph{Zea mays} ssp. \emph{mexicana} (referred to \emph{mexicana} hereafter) and Mexican highland maize have become a well-studied pair (although ssp. \emp{parvaglumis} has also been shown to hybridize with maize, \cite{wilkes1977hybridization}).

Introgression between \emph{mexicana} and Mexican highland maize has been reported based on evidence from both morphological data \cite{wilkes1977, lauter2004, doebley1984} and molecular analysis \cite{matsuoka2002, vanHeerwaarden2011, doebley1987, warburton2011, fukunaga2005}. [[Li's and my bib tags might be in conflict, e.g. wilkes1977 is likely my wilkes1977hybridization.]]
.
 Doebley (\cite{doebley1987patterns}) found that north-central high-elevation Mexican landraces (Apachito, Arrocillo Amarillo, Azul, Celaya, Chaloqueno, Conico, Conico Norteno, Gordo, Harinoso de Ocho Occidentales)[[special characters]] show evidence of introgression of two alleles from \emph{mexicana}, and suspected that they might have fitness advantages for adapting to highland environmental conditions.

Population structure analyses by \cite{matsuoka2002single} estimate that \emph{mexicana} gene flow has contributed to the gene pools of the Cacahuacintle, Palomero de Jalisco, and Palomero Taloqueno landraces, but also into (non-landrace) Mexican maize at the same elevation.

However, \cite{hufford2013} for the first time revealed the evidence of adaptive introgression from \emph{mexicana} to Mexican highland maize.
The authors identified nine genomic regions, which showed evidence of introgression from \emph{mexicana} to maize in both the HAPMIX and the linkage model of STRUCTURE analyses with over seven sympatric population pairs among the total nine pairs sampled. 
Among the nine regions, three spans the centromeres of chromosomes 5, 6, and 10, and one is located in the inversion polymorphism on chromosome 4, suggesting a significant role of genome structures restricting recombination in adaptive introgression.
By further characterizing the nine introgression regions, it is found that most regions contain long tracts of zero diversity, enriched with QTL linked with anthocyanin content and leaf macrohairs \cite{lauter2004} and over-represented with the SNPs demonstrating high association with temperature seasonality.
Growth chamber experiments with maize populations with introgression from \emph{mexicana} on chromosome 4 (associated with QTL controlling pigment density and macrohairs) and 9 (overlapped with QTL for macrohairs) exhibited more macrohairs and greater pigmentation under the highland environmental settings than the populations with absence of introgression from \emph{mexicana}.

Gene flow between \emph{mexicana} and maize is low enough that only fitness-neutral or advantageous alleles are likely to be passed \cite{slatkin1987gene}.

From \cite{baltazar2005pollination}
Gene flow is probably greater from teosinte to maize.
Certain morphological and flower timing characterists promote genetic isolation between maize and teosinte and slow rates of introgression.
Gene flow from maize to teosinte occurs most easily when teosinte pollinates maize
[[what paper can i cite to mention the gene tb1 and its role in hybridization, viz a viz restricting directionality of teosinte pollen into maize

From \cite{jiang1999genetic}, {bitocchi2009introgression}
Maize-maize introgression
Research on introgression between highland and lowland maize shows adaptive traits being transferred

\cite{bitocchi2009introgression} has a good review of the history of domestication, introduction to Europe, and further development.


\subsection*{wheat}

Domesticated einkorn wheat (Triticum monococcum monococcum), wild relative einkorn wheat {T. m. boeoticum), and the progenitor T. m. boeoticum from the Karacadag [[special character]] mountains of southeast Turkey \cite{heun1997site}.
Wild emmer wheat (Triticum turgidum subsp. dicoccoides), the progenitor of modern (tetraploid and hexaploid) wheat, is from the Fertile Crescent \cite{lev2000cradle}.
Two distinct taxa of Triticum dioccoides exist (a western one and a central-eastern one), but the central-eastern one was the progenitor species from which modern maize was domesticated \cite{ozkan2005reconsideration}.
Localizing domestication centers for other forms of wheat at a resolution greater than the Fertile Crescent is not yet possible \cite{nesbitt1998wheat}, but then \cite{lev2000cradle} point to "a small core area within the Fertile Crescentâ€”near the upper reaches of the Tigris and Euphrates rivers [HN6] in present-day southeastern Turkey/northern Syria".
So, wheat has one restricted center of domestication, with wild relatives nearby.  From \cite{ozkan2005reconsideration}, "The geographical distribution of T. dicoccoides reported by Zohary and Hopf (2000) includes the western Fertile Crescent, the central part of southeastern Turkey and areas in eastern Iran and Iraq. Johnson (1975) reports that from southeastern Turkey to Iran and Iraq the species is progressively substituted by the wild tetraploid wheat T. araraticum. T. dicoccoides also grows on the basaltic rocky slopes of the Karacadag mountains in southeastern Turkey (Johnson 1975; Harlan and Zohary 1966).
The a, b, c, and f alleles of mildew resistance gene Pm3 may have been introgressed into wheat from wild emmer wheat shortly after domestication (although the e, d, and g alleles were probably formed by de novo mutations) \cite{TPJ:TPJ2772}.

The following are just notes.

From {label24}
review/overview.  See table 1

From {label25}
Wheat was given leaf rust resistance genes from wild populations (summarized in Table 1)

From {label26}
A few examples (wild genes into wheat) given.  Originally, only with cross-compatible species, now with cross-incompatible species.
Cites and tests several QTLs

From {label31}
"suggestions are made concerning techniques for exploitation of the wild diploid species in wheat breeding programs."

\subsection*{barley}

Polyphyletic \cite{azhaguvel2007phylogenetic}.
\cite{von1995ecogeographical} states that barley and its progenitor are the same species, as they are not sufficiently genetically distinct and geneflow continues unhindered.
The progenitor species is Hordeum spontaneum \cite{oka2012origin}.

Notes:

From {label24}
review/overview.  See table 1

From {label30}
"wild barley does harbour valuable alleles, which can enrich the genetic basis of cultivated barley and improve quantitative agronomic traits."

\subsection*{sunflower}

From {label33}
yeah, introgression with wild relatives may be common, unsure about direction
intermediates found in the field

\subsection*{rice}

The center of rice domestication is not known with confidence, with evidence pointing towards both China and India \cite{oka2012origin} [[see the full review on page 16]].
There are main cultigens of rice; common rice, Oryza sativa, and African rice, O. glaberrima, which are easily distinguish by ligule length, number of secondary panicle branches, panicle axis thickness, and because O. glaberrima is annual \cite{oka2012origin}.
Oryza sativa was domesticated from wild O. rufipogon "or the Asian form of O. perennis complex" \cite{oka2012origin}, whereas O. glaberrima was domesticated from O. breviligulata \cite{oka2012origin}.
Both O. sativa/O. rufipogon and O. glaberrima/O.breviligulata produce weedy hybrids.
The greater genetic diversity within O. sativa is likely due to introgression with wild relatives both during domestication and upon the dispersal of O. sativa into new environments and sympatry with new relatives \cite{second1982origin}.
Several resistance genes (grassy stunt virus, bacterial blight, brown planthopper, blast) have been introgressed from wild relatives into O. sativa L. by researchers \cite{brar1997alien, khush1974inheritance}, but adaptive introgression has also happened without human intervention \cite{second1982origin}.

\subsection*{potato}

Modern Solanum tuberosum was domesticated in southern Peru from northern members of the polyphyletic S. brevicaule complex \cite{spooner2005single}.
These northern members of the S. breviaule complex are not clearly defined, and may in fact be one singular species (in which case the species name would be S. bukasovii) \cite{spooner2005single}.
Resistance genes have been experimentally introgressed from wild poptato relatives (S. tuberosum subsp. andigena, \cite{van1999tight}, S. bulbocastanum, \cite{van2003ancient}) into cultivated potato, although this process is hindered by compatability issues including ploidy levels and Endosperm Balance Number \cite{johnston1980significance}.

\subsection*{tomato}

The domestication center for tomato is not known with certainty, but \cite{sims1979history} quotes Dr. Charles Rick (University of California, August 1978) stating that tomato was domesticated from Lycopersicon esculentum v. cerasiformae in the New World (most likely Mexico).
Most of the genetic diversity resides within the wild relatives of the tomato \cite{rick1988tomato} \cite{miller1990rflp}, so breeders have long turned to these plants for their germplasm \cite{rick1982potential}.
\cite{hanson2000mapping} mapped the tomato yellow leaf curl virus resistance gene in L. hirsutum f. glabratum accession 'B 6013' (line H24) (from \cite{banerjee1990transfer}) to an introgression on chromosome 11.

\subsection*{common bean}

\subsection*{Other Notes}

\cite{zohary2012domestication} has Map 1, showing the domestication center of einkorn wheat, emmer wheat, barley, chickpea, flax, lentil, pea, and bitter vetch in the Fertile Crescent.
Most of the wild progenitors of founder crops (excluding barley and flax) have a distribution restricted to the Fertile Crescent \cite{zohary2012domestication}.
However, such exceptions as barley, flax, foxtail millet, and oat have wild progenitors which inhabit a wide range of habitats beyond the Fertile Crescent.

\cite{jarvis1999wild} Table 2 is an incredible resource, cited examples of farmer selection and/or use of introgressed types, and confirmation of introgression

https://books.google.com/books?hl=en&lr=&id=CvcbUopfa54C&oi=fnd&pg=PA7&dq=common+bean+domestication&ots=ggAF-EoWG8&sig=dI9wOyATpzAelxLf_cg-gs0J2gI#v=onepage&q=common%20bean%20domestication&f=false

\subsection*{non-crop systems}
From {label3}
Bacteria, malaria mosquitoes, blackflies, Darwin's finches, butterflies (each example has a cited source, not included in the bibliography of this outline).
Also, non-crop plants, like trees (oak, larch).

From {label15}
Helianthus, Iris
Mouse, Salamander

From {label21}
Iris, Helianthus, Cowania/Purshia
Dacus (now Bactrocera), Anopheles
Haemophilus influenza, Trypanosoma cruzi

From {label23}
165 proposed cases of introgression, 65 "deemed to be sufficiently documented" (plants)

From {label28}
Milkweeds

From {label29}
White clover












\begin{thebibliography}{1}
\bibitem{label1} H. G. Wilkes (1977) 
Hybridization of Maize and Teosinte, in Mexico and Guatemala and the Improvement of Maize. 
{\em Economic Botany\} 31:254-293

\bibitem{label2} C. Lumholtz (1902)
Unknown Mexico. Charles Scribner's Sons, New York. Vol I. 530 pp.
%This citation was problematic

\bibitem{label3} J. Mallet (2005)
Hybridization as an invasion of the genome.
{\em TRENDS in Ecology and Evolutin\} 20:229-237

\bibitem{label4} J. Ross-Ibarra, M. Tenaillon, B. S. Gaut (2009)
Historical Divergence and Gene Flow in the Genus Zea.
{\em Genetics\} 181:1399-1413

\bibitem{label5} N. C. Ellstrand, H. C. Prentice, J. F. Hancock (1990)
Gene Flow and Introgression from Domesticated Plants into their Wild Relatives.
{\em Annual Review of Ecology and Systematics\} 30:539-563

\bibitem{label6} D. Quist, I. H. Chapela (2001)
Transgenic DNA introgressed into traditional maize landraces in Oaxaca, Mexico.
{\em Nature\} 414:541-543
%RETRACTED?

\bibitem{label7} J. Doebley (1990)
Molecular Evidence for Gene Flow among {\em Zea\} Species.
{\em BioScience\} 40:443-448

\bibitem{label8} J. F. Doebley (1984)
Maize Introgression into Teosinte - A Reappraisal.
{\em Annals of the Missouri Botanical Garden\} 71:1100-1113

\bibitem{label9} J. Doebley (1990)
Molecular Evidence and the Evolution of Maize.
{\em Economic Botany\} 44:6-27
%a review paper

\bibitem{label10} J. Doebley, M. M. Goodman, C. W. Stuber (1987)
Patterns of isozyme variation between maize and Mexican annual teosinte.
{\em Economic Botany\} 41:234-246

\bibitem{label11} B. M. Baltazar, J. de J. Sanchez-Gonzalez (2005)
Pollination between maize and teosinte: an important determinant of gene flow in Mexico
{\em Theoretical Applied Genetics\} 110:519-526
%special character for the Sanchez name, plus how to abbreviate "de"

\bibitem{label12} M. L. Warburton, G. Wilkes, S. Taba, A. Charcosset, C. Mir, F. Dumas, D. Madur, S. Dreisigacker, C. Bedoya, B. M. Prasanna, C. X. Xie, S. Hearne, J. Franco (2011)
Gene flow among different teosinte taxa and into the domesticated maize gene pool.
{\em Genetic Resources and Crop Evolution\} 58:1243-1261

\bibitem{label13} C. Jiang, G. O. Edmeades, I. Armstead, H. R. Lafitte, M. D. Hayward, D. Hoisington (1999)
Genetic analysis of adaptation of differences between highland and lowland tropical maize using molecular markers.
{\em Theoretical Applied Genetics\} 99:1106-1119

\bibitem{label14} E. Bitocchi, L. Nanni, M. Rossi, D. Rau, E. Bellucci, A. Giardini, A. Buonamici, G. G. Vendramin, R. Papa (2009)
Introgression from modern hybrid varieties into landrace populations of maize ({\em Zea mays\} ssp. {\em mays\} L.) in central Italy.
{\em Molecular Ecology\} 18:603-621

\bibitem{label15} M. L. Arnold, N. H. Martin (2009)
Adaptation by introgression.
{\em Journal of Biology\} 8:82
%mini review of introgression

\bibitem{label16} N. C. Ellstrand, C. Garner, S. Hegde, R. Guadagnuolo, L. Blancas (2007)
Spontaneous hybridization between Maize and Teosinte.
{\em Journal of Heredity\} 98:183-187

\bibitem{label17} M. B. Hufford, P. Lubinsky, T. Pyhajarvi, M. T. Devengenzo, N. C. Ellstrand, J. Ross-Ibarra (2013)
The genomic signature of crop-wild introgression in maize.
{\em PLOS Genetics\} 9:
%special characters, page numbers?

\bibitem{label18} Y. Matsuoka, Y. Vigouroux, M. M. Goodman, J. Sanchez G., E. Buckler, J. Doebley (2002)
A single domestication for maize shown by multilocus microsatellite genotyping.
{\em PNAS\} 99:6080-6084

\bibitem{label19} J. van Heerwaarden, J. Doebley, W. H. Briggs, J. C. Glaubitz, M. M. Goodman, J. de Jesus Sanchez Gonzalez, J. Ross-Ibarra (2010)
Genetic signals of origin, spread, and introgression in a large sample of maize landraces.
{\em PNAS\} 108:1088-1092
%Gonzalez, "van"

\bibitem{label20} H. G. Wilkes (1970)
Teosinte introgression in the maize of the Nobogame Valley
{\em Botanical Museum Leaflets\} 22:297-311

\bibitem{label21} M. L. Arnold (2004)
Transfer and origin of adaptations through natural hybridization: Were Anderson and Stebbins Right?
{\em The Plant Cell\} 16:562-570
%introgression review

\bibitem{label22} C. N. Stewart Jr., M. D. Halfhill, S. Warwick (2003)
Transgene introgression from genetically modified crops to their wild relatives.
{\em Nature\} 4:806-817
%is nature reviews the same thing as nature?
%good review of introgression with crops
%Table 1 shows crops and their progenitor species and main wild relatives

\bibitem{label23} L. H. Rieseberg, J. F. Wendel (1993)
Hybrid zones and the evolutionary process.
{\em Journal of Evolutionary Biology\} 7:631-634

\bibitem{label24} E. Nevo, G. Chen (2010)
Drought and salt tolerances in wild relatives for wheat and barley improvement.
{\em Plant, Cell Environment\} 33:670-685

\bibitem{label25} E. Autrique, R. P. Singh, S. D. Tanksley, M. E. Sorrells (1995)
Molecular markers for four lef rust resistance genes introgressed into wheat from wild relatives
{\em Genome\} 38:75-83

\bibitem{label26} S. G. Hegde, J. G. Waines (2004)
Hybridization and introgression between bread wheat and wild and weedy relatives in North America.
{\em Crop Science\} 44:1145-1155

\bibitem{label27} M. A. Chapman, J. M. Burke (2006)
Letting the gene out of the bottle: The population genetics of genetically modified crops.
{\em New Phytologist\} 170:429-443

\bibitem{label28} S. B. Broyles (2002)
Hybrid bridges to gene flow: A case study in milkweeds ({\em Asclepias\}).
{\em Evolution\} 56:1943-1953


\bibitem{label29} S. W. Hussain, W. M. Williams (1997)
Development of a fertile genetic bridge between {\em Trifolum ambiguum\} M. Bieb. and {\em T. repens\} L.
{\em Theoretical Applied Genetics\} 95:678-690

\bibitem{label30} M. von Korff, H. Wang, J. Leon, K. Pillen (2006)
AB-QTL analysis in spring barley: II. Detection of favourable exotic alleles for agronomic traits introgressed from wild barley ({\em H. vulgare\} ssp. {\em spontaneum\}).
{\em Theoretical Applied Genetics\} 112:1221-1231
%special characters

\bibitem{label31} D. Zohary, J. R. Harlan, A. Vardi (1969)
The wild diploid progenitors of wheat and their breeding value.
{\em Euphytica\} 18:58-65

\bibitem{label32} [[A huge fricken list of names]] (2013)
Hybridization and speciation.
{\em Journal of Evolutionary Biology\} 26:229-246

\bibitem{label32} K. Fukunaga, J. Hill, Y. Virouroux, Y. Matsuoka, J. Sanchez G., K. Liu, E. S. Buckler, J. Doebley (2005)
Genetic diversity and Population Structure of Teosinte.
{\em Genetics\} 169:2241-2254

\bibitem{label33} M. Reagon, A. A. Snow (2006)
Cultivated {\em Helianthus annus\} (Asteraceae) volunteers as a genetic "bridge" to weedy sunflower populations in North America.
{\em Ammerican Journal of Botany\} 93:127-133

\bibitem{label34} J. A. Able, P. Langridge, A. S. Milligan (2006)
Capturing diversity in the cereals: many options but little promiscuity.
{\em TRENDS in Plant Science\} 12:1360-1385

\end{thebibliography}{1}

\end{document}
