\documentclass[11pt]{article}
\usepackage{color}
\usepackage{authblk}%allows footnote format for authors
\usepackage[letterpaper, margin=1in]{geometry} %package that allows changes in margins and header/footers

\newcommand{\mbh}[1]{\textcolor{green}{ \emph{\scriptsize  #1}} } %creating command for Matt's comments
\newcommand{\lwang}[1]{\textcolor{red}{ \emph{\scriptsize  #1}} } %creating command for Li's comments

\title{Crop adaptation through gene flow}



\begin{document}

\maketitle

\section*{Outline}
\subsection*{Outline parts}

\subsection*{Introduction}
 "Whenever sympatric populations of crops and their wild relatives show a similar trait, there will be several possible explanations: (1) retention of an ancestral trait, (2) convergent or parallel evolution, and (3) introgression.  Perhaps even more difficult is establishing the direction of introgression; does gene flow go from crop into wild species, the reverse, or in both directions?" and "For this reason, introgression may be best studied between crops and their more distant wild relatives where the probability of finding taxon-specific genetic markers is increased." \cite{doebley1990molecular}


1\subsection*{Introgression in Crops (General)}

From {stewart2003transgene}
A very good review.
Hybrid zones make for a great model system of study.
Introgression from wild populations into agronomic populations is potentially beneficial for breeding programmes.
It is generally easier to introgress into crops than introgressing into wild populations (stronger selection, overcoming LD).

\cite{harlan1966distribution}\ The premise; domestication takes place at the edges of a progenitor's range, where it barely grows.  there, farmers work on the crop to make it more adapted.  if this is the case, this could maintain progenitor populations long after domestication, and may increase gene flow between the original progenitor population and the expanding domesticate.  This is probably unlikely to be advantageous for the domesticate, however.

From {label32}
Another very good review, with more of a molecular emphasis

From {label34}
The difficulties of utilizing wild relatives in breeding programmes effectively.

From {label6} {label7} {label26} {Label27}
Lots of papers focus on gene flow from crop to wild relative, particularly about transgene escape

\subsection*{Maize}

Maize is a classical model in crop species to study the issue of adaptive introgresssion from wild relatives, as it still grows in sympatry with many of its wild relatives \cite{hufford2013}.

Maize was domesticated about 10,000 years ago in Mexico \cite{smith1997initial} \cite{hufford2012comparative} in the Balsas River valley \cite {matsuoka2002single} from teosinte.

Mexican farmers have long been aware of maize-teosinte hybrids, and in some places, such as the Nobogame valley, farmers have observed desirable traits passed to their crops through the hybrid plants \cite{wilkes1977hybridization} \cite{lumholtz1902unknown} \cite{wilkes1970teosinte}.

In particular, \emph{Zea mays} ssp. \emph{mexicana} (referred to \emph{mexicana} hereafter) and Mexican highland maize have become a well-studied pair (although ssp. \emp{parvaglumis} has also been shown to hybridize with maize, \cite{wilkes1977hybridization}).

Introgression between \emph{mexicana} and Mexican highland maize has been reported based on evidence from both morphological data \cite{wilkes1977, lauter2004, doebley1984} and molecular analysis \cite{matsuoka2002, vanHeerwaarden2011, doebley1987, warburton2011, fukunaga2005}. [[Li's and my bib tags might be in conflict, e.g. wilkes1977 is likely my wilkes1977hybridization.]]
.
 Doebley (\cite{doebley1987patterns}) found that north-central high-elevation Mexican landraces (Apachito, Arrocillo Amarillo, Azul, Celaya, Chaloqueno, Conico, Conico Norteno, Gordo, Harinoso de Ocho Occidentales)[[special characters]] show evidence of introgression of two alleles from \emph{mexicana}, and suspected that they might have fitness advantages for adapting to highland environmental conditions.

Population structure analyses by \cite{matsuoka2002single} estimate that \emph{mexicana} gene flow has contributed to the gene pools of the Cacahuacintle, Palomero de Jalisco, and Palomero Taloqueno landraces, but also into (non-landrace) Mexican maize at the same elevation.

However, \cite{hufford2013} for the first time revealed the evidence of adaptive introgression from \emph{mexicana} to Mexican highland maize.
The authors identified nine genomic regions, which showed evidence of introgression from \emph{mexicana} to maize in both the HAPMIX and the linkage model of STRUCTURE analyses with over seven sympatric population pairs among the total nine pairs sampled. 
Among the nine regions, three spans the centromeres of chromosomes 5, 6, and 10, and one is located in the inversion polymorphism on chromosome 4, suggesting a significant role of genome structures restricting recombination in adaptive introgression.
By further characterizing the nine introgression regions, it is found that most regions contain long tracts of zero diversity, enriched with QTL linked with anthocyanin content and leaf macrohairs \cite{lauter2004} and over-represented with the SNPs demonstrating high association with temperature seasonality.
Growth chamber experiments with maize populations with introgression from \emph{mexicana} on chromosome 4 (associated with QTL controlling pigment density and macrohairs) and 9 (overlapped with QTL for macrohairs) exhibited more macrohairs and greater pigmentation under the highland environmental settings than the populations with absence of introgression from \emph{mexicana}.

Gene flow between \emph{mexicana} and maize is low enough that only fitness-neutral or advantageous alleles are likely to be passed \cite{slatkin1987gene}.

From \cite{baltazar2005pollination}
Gene flow is probably greater from teosinte to maize.
Certain morphological and flower timing characterists promote genetic isolation between maize and teosinte and slow rates of introgression.
Gene flow from maize to teosinte occurs most easily when teosinte pollinates maize
[[what paper can i cite to mention the gene tb1 and its role in hybridization, viz a viz restricting directionality of teosinte pollen into maize

From \cite{jiang1999genetic}, {bitocchi2009introgression}
Maize-maize introgression
Research on introgression between highland and lowland maize shows adaptive traits being transferred

\cite{bitocchi2009introgression} has a good review of the history of domestication, introduction to Europe, and further development.

\cite{hammer1987collection}\ cites an example in Italy of one domesticated maize adapting to coastal environment by introgressions from another domesticated maize.

\subsection*{wheat}

The first domesticated wheat was einkorn wheat (Triticum monococcum), domesticated from wild einkorn (T. m. boeoticum) in or around the Karacadag mountains of southeast Turkey \cite{heun1997site, harlan1966distribution, ozkan2002aflp}\.
Subsequently, emmer wheat (Triticum dicoccum) was domesticated from hybrids of diverse populations of wild emmer wheat (T. dioccoides) in the Fertile Crescent \cite{lev2000cradle, civavn2013reticulated, luo2007structure}.
Emmer wheat is tetraploid, and hybridization with wild diploid Tausch's goatgrass (Aegilops tauschii) lead to the production of hexaploid wheat \cite{salamini2002genetics, hancock2012plant, dvorak2006molecular}\.
Emmer wheat has high genetic diversity, likely due to gene flow from wild emmer \cite{luo2007structure, dvorak2006molecular}\.

Durum wheat (T. turgidum ssp. durum) was developed from emmer wheat as well.

Common bread wheat, T. aestivum, is an allohexaploid with chromosome sets from red wild einkorn wheat (T. urartu), Aegilops speltoides, and Aegilops tauschii \cite{mcfadden1946origin, petersen2006phylogenetic}\, and furthermore, each of these species is the result of ancient hybridizations \cite{marcussen2014ancient}\.
Hexaploidization probably occured twice, in southeast Turkey/northern Syria and, more recently, Iran \cite{giles2006gludy}\.

Domesticated einkorn wheat (Triticum monococcum monococcum), wild einkorn wheat (T. m. boeoticum), and the progenitor T. m. boeoticum are from the Karacadag [[special character]] mountains of southeast Turkey \cite{heun1997site}.
Wild emmer wheat (Triticum turgidum subsp. dicoccoides), the progenitor of modern (tetraploid and hexaploid) emmer wheat, is from the Fertile Crescent \cite{lev2000cradle, ozkan2002aflp}.
Two distinct taxa of Triticum dioccoides exist (a western one and a central-eastern one), but the central-eastern one was the progenitor species from which modern wheat was domesticated \cite{ozkan2005reconsideration}.
Localizing domestication centers for other forms of wheat at a resolution greater than the Fertile Crescent is not yet possible \cite{nesbitt1998wheat}, but then \cite{lev2000cradle} point to "a small core area within the Fertile Crescentâ€”near the upper reaches of the Tigris and Euphrates rivers [HN6] in present-day southeastern Turkey/northern Syria".
So, wheat has one restricted center of domestication, with wild relatives nearby.
From \cite{ozkan2005reconsideration}\, "The geographical distribution of T. dicoccoides reported by Zohary and Hopf (2000) includes the western Fertile Crescent, the central part of southeastern Turkey and areas in eastern Iran and Iraq. Johnson (1975) reports that from southeastern Turkey to Iran and Iraq the species is progressively substituted by the wild tetraploid wheat T. araraticum. T. dicoccoides also grows on the basaltic rocky slopes of the Karacadag mountains in southeastern Turkey (Johnson 1975; Harlan and Zohary 1966).

The progressive domestication of wheat has involved many introgressive acts between geographically separated lineages, evinced by identification of genomes' various sources in wheats with elevated ploidy levels [[see the two papers i included for bread wheat background]]

The a, b, c, and f alleles of mildew resistance gene Pm3 may have been introgressed into wheat from wild emmer wheat shortly after domestication (although the e, d, and g alleles were probably formed by de novo mutations) \cite{TPJ:TPJ2772}.
Stem rust resistance bred into cultivated T. durum (tetraploid) from wild einkorn wheat \cite{gerechter1971transfer}\.
Pm36, a novel podery mildew resistance gene, was also experimentally introgressed from T. turgidum var. dicoccoides into durum wheat \cite{blanco2008molecular}\.

\subsection*{barley}

Domesticated and wild barleys belong to the same species, Hordeum vulgare, and are biologically capable of producing viable offpsring via hybridizaion \cite{von1995ecographical}.
Barley (Hordeum vulgare subsp. vulgare) is believed to have been domesticated from wild subsp. spontaneum in the Fertile Crescent and by subsp. spontaneum var. agriocrithon in Tibet roughly 10,000 years ago \cite{takahashi1955origin, badr2000origin, oka2012origin, azhaguvel2007phylogenetic, haberer2015barley}\, but many of the details of barley domestication are still disputed.

Presently, the distribution of spontaneum consists of the Mediterranean, the Middle-East, and West-Central Asia, while other barley wild relatives inhabit Asiatic regions (including Tibet) more broadly \cite{nevo2010drought, harlan1995living}\.
These wild relatives inhabit regions spanning such abiotic clines as temperature, precipitation, soil type, and altitude, as well as biotic clines \cite{nevo2010drought}\.

The barley domestication process has reduced the number of alleles in the domesticate to only 40% of that found in wild barley, though there remains a great deal of phenotypic diversity among the wild barleys \cite{ellis2000wild}\.

Variety agriocrithon (H. vulgare ssp. vulgare f. agriocrithon) is genetically diverse, and is found throughout much of the range of Barley.
Some have suggested that agriocrithon may be the progenitor of six-rowed barley, the product of a hybridization between eastern and western cultivated barley, or a wild-domesticate hybrid \cite{staudt1961origin, zohary1959hordeum, murphy1982origin}\, but \cite{azhaguvel2007phylogenetic} dispute these theories, suggesting instead that it arose from ssp. spontaneum, perhaps more than once (in Israel as well as in Tibet).
Wild-domesticate breeding experiments have shown that wild barleys have alleles for several important agronomic phenotypes, including brittleness, flowering time, plant height, lodging, and yield,  \cite{von2006ab, handley1994chromosome}\.
See art%0223-4 for list of papers that discuss the beneficial effect of exotic genes on biotic, abiotic, and quality traits.

Oweing to the lack of evidence of adaptive introgression between wild and domesticated barley is the low rate of outcrossing of spontaneum, estimated by \cite{brown1978outcrossing} to be 1.6% in Israeli populations.

Barley/H. spontaneum hybrids are fertile, but hybrids of other wild relatives generally are not.
There has been little genetic investigation into spontaneous barley/spontaneum hybrids \cite{ellstrand2003dangerous}

Look, there's no barrier to hybridization, there was a broad suite of wild barleys already in place, they're all locally adapted, this is a prime example of POSSIBLE introgression, but where's the evidence?

Introgression from wild to domesticate has been shown to happen over distances of more than a kilometer \cite{hillman2001new}\.

\subsection*{sunflower}

Helianthus annuus shows evidence of domestication in eastern United States \cite{harter2004origin, wills2006chloroplast}\, with additional evidence of a possible second origin of domestication in Mexico \cite{lentz2008sunflower}\.
Helianthus is a versatile species, capable of adapting to a wide range of environments across the Americas, Europe, Asia, and Australia \cite{kane2008genetics}\.
[[This versatility may be due in part either to phenotypic plasticity or genetic adaptability \cite{maron2004rapid}\.]] This sentence may not help much, actually.  We're betting on genetic adaptability via introg hyb

H. annus adapted to the environment in Texas by hybridizing with H. debilis ssp. cucumerifolius, gaining advantageous alleles (these hybrids are now called H. annus ssp. texanus, \cite{kim1999genetic, heiser1951hybridization, rieseberg1999hybrid, rieseberg1990helianthus}.
Helianthus has several genes for downy mildew resistance.
Each imparts resistance to one or more races of P. halstedii, one of the most agronomically important diseases in sunflower cultivation \cite{cohen1973factors}\.
Some of these downy mildew resistence genes were found in wild relatives (including H. argophyllus, H. tuberosus, and H. praecox) and have been successfully bred into modern H. annus \cite{miller1991inheritance}\.
PlArg, an allele found in wild silverleaf sunflowers (Helianthus argophyllus, inbred line Arg1575-2), confers resistance to all known (20 or more) races of downey mildew \cite{dussle2004pl}\, while others (Pl1-Pl11) are effective for one or more types \cite{rahim2002inheritance}\.
H. annus lives in sympatry with wild relatives like H. petiolaris and H. bolanderi and forms  stable hybrid populations \cite{schwarzbach2002likely, rieseberg1988molecular, welch2002patterns}\.
H. annus shows signs of persistent introgressive hybridization with H. petiolaris with evidence of positive selection driving some of the genetic differentiation between the two species \cite{yatabe2007rampant}\.
\cite{kane2008genetics} finds that most gene flow in weedy Helianthus is within geographic regions, rather than between separate popultions.
Wild sunflowers are locally adapted, and weedy hybrid populations share these adaptations \cite{kane2008genetics}\.
Weediness is a trait that has likely evolved many times \cite{kane2008genetics}\.

Silverleaf sunflower has also been the focus of drought resistance breeding efforts \cite{saucaÌ†2010introgression}\ and Phomopsis resistance breeding efforts \cite{besnard1997specifying}\.

\subsection*{rice}

There are two main cultigens of rice; Asian rice, Oryza sativa, and African rice, O. glaberrima, which are easily distinguish by ligule length, number of secondary panicle branches, panicle axis thickness, and because O. glaberrima is annual \cite{oka2012origin}.

The centers of rice domestication are not known with complete confidence, but genetic and archaeobotanical evidence points towards both the Yangzee Basin in China and the Ganges plains in India for O. sativa, 8,200-13,500 years ago, from wild O. rufipogon  \cite{oka2012origin, fuller2010consilience, ricepedia, molina2011molecular}\ and the Upper Niger River delta in Mali, Africa for O. glaberrima, 2-3,000 years ago, from ancestor O. barthii \cite{ricepedia}\.

The two main subspecies of Asian rice, indica and japonica, were domesticated from wild O. rufipogon "or the Asian form of O. perennis complex" \cite{oka2012origin}\ or O. nivara, which is closely related to O. rufipogon \cite{sharma2000origin, oka1988origin}\, whereas O. glaberrima was domesticated from O. breviligulata \cite{oka2012origin}\.

Both O. sativa/O. rufipogon and O. glaberrima/O.breviligulata produce weedy hybrids.
The greater genetic diversity within O. sativa is likely due to introgression with wild relatives both during domestication and upon the dispersal of O. sativa into new environments and sympatry with new relatives \cite{second1982origin}.

O. sativa has two main subspecies, Indica (with subpopulations indica and aus) and Japonica (with subpopulations temperate japonica, tropical japonica known as javanica, and aromatic) \cite{chang2003origin, glaszmann1987isozymes, ricepedia}.
These subspecies' display adaptations to the environmental coniditions corresponding to the differentiated geographical locations they inhabit \cite{khush2003classifying}\.

There are about 20 other wild species within the genus Oryza \cite{ricepedia}\.

There exists evidence of gene flow between rice subpopulations and into wild rice \cite{zhao2010genomic}\.

Several resistance genes (grassy stunt virus, bacterial blight, brown planthopper, blast) have been introgressed from wild relatives into O. sativa L. by researchers \cite{brar1997alien, khush1974inheritance}, but adaptive introgression has also happened without human intervention \cite{second1982origin}.


\subsection*{rye}

Rye (Secale cereale) was most likely domesticated from S. vavilovii \cite{stutz1972origin}\.
The species Secale cereale contains domesticated, weedy, and wild rye subspecies \cite{khush1961cytogenetic}\.
Rye domestication has recieved comparatively little academic interest \cite
N. I. Vavilov (the namesake of the species name of this plant) hypothesized that rye originated as a secondary crop during the domestications of wheat, barley, and/or peas \cite{vavilov1928geographical}\.

\subsection*{potato}

Modern potato Solanum tuberosum is believed to have been domesticated in southern Peru in sympatry with a multitude of wild relatives about 6000 years ago, although the exact location and formal classification and phylogenetic relationships between these taxa have long been disputed \cite{huaman2002reclassification, spooner2005single, pickersgill1977origins, hawkes1988evolution}\.
The northern members of the polyphyletic S. brevicaule wild potato complex have been identified as likely progenitors of modern potatoes \cite{correll1962potato}, but determination of a single progenitor species is unlikely, either because widespread gene flow in the complex will mask signs of this phylogeny or because s. tuberosum has a polyphyletic origin.
Then again, these northern members of the S. breviaule complex are not clearly defined, and may in fact be one singular species (in which case the species name would be S. bukasovii) \cite{spooner2005single}.

Although potatoes are usually propogated clonally, farmers also promote sexyal hybridization at times to improve disease resistance and develop new cultivars \cite{quiros1992increase}\.
Farmers continue to grow potatoes in close proximity to wild relatives, resulting in domesticate-weedy-wild hybrid complexes which promote introgressive hybridizatiion \cite{rabinowitz1990high, johns1987relationships, linder1987diversity}\.

The various cultivars of Andean potatoes are interfertile, forming one large plastic gene pool \cite{quiros1992increase}\.
Andean potatoes exhibit high ecological versatility, due in part either to alleleic diversity in polyploids or introgression of desirable alleles from wild relatives in diploids \cite{zimmerer1998ecogeography}\.

Contrary to expectations \cite{hamrick1983distribution}\, landrace tetraploid potatoes show more genetic diversity than diploids \cite{zimmerer1991geographical}\.

Widespread introgression and hybridization in potatoes \cite{grun1990evolution}\.

, potato landraces show a weaker correlation, due perhaps to confounding influences of farmer interactions (including environmental manipulation and artificial selection by farmers \cite{zimmerer1991geographical, [[cite other sources from page 45 in this paper}


These complexes, combined with a diverse range of biotic and environmental selective pressures and local farming practices (human-mediated migration, isolated farmsteads in fertile valleys, clonal propogation, and intentional maintenance of a variety of landraces), have fostered expansion of genetic diversity within potatoes subsequent to domestication \cite{brush1995potato}\.
However, as farmers tend to abandon fields after being used for potato cultivation, it is less likely that hybrids have an opportunity to form stable populations for maintained introgressive gene flow into the domesticated potato gene pool \cite{brush1995potato}\.

Natural selection may be subverted by selection and preference of farmers \cite{brush1981dynamics}\.

Drought- and frost-resistance genes introgressed from wild relatives expediated the spread of cultivated potatoes into the Central Andean Altiplano \cite{johns1986ongoing, hawkes1962origin, schmiediche1980breeding}\.

Cultivated diploid potatoes have been shown introgressed with genes from S. sparsipilum \cite{rabinowitz1990high}\ and S. megistacrolobum \cite{johns1987relationships, huaman1980solanum}.

Resistance genes have been experimentally introgressed from wild poptato relatives (S. tuberosum subsp. andigena, \cite{van1999tight}, S. bulbocastanum, \cite{van2003ancient}) into cultivated potato, although this process is hindered by compatability issues including ploidy levels and Endosperm Balance Number \cite{johnston1980significance}.

\subsection*{tomato}

The domestication center for tomato is not known with certainty, but \cite{sims1979history} quotes Dr. Charles Rick (University of California, August 1978) stating that tomato was domesticated from Lycopersicon esculentum v. cerasiformae [[solanum lycopersicum cerasiforme]] in the New World (most likely Mexico) [[http://www.landscapeimagery.com/tomato.html]].
\cite{robertson2007genetic, bai2007domestication} and [[http://www.landscapeimagery.com/tomato.html]] agrees.
\cite{nesbitt2002comparative}, however, conclude that v. cerasiformae is not the direct ancestor of the domesticated tomato, but rather a hybrid between wild tomatoes L. esculentum and L. pimpinellifolium.
Widescale recent introgression between wild relatives confound attempts at elucidating phylogenies.
Since the work of Charles Rick in 1940s and 50s \cite{rick1953novel}\, breeding efforts have focused on incorporating and utilizing the relatively diverse genetic germplasm of wild tomato relatives \cite{rick1988tomato} \cite{miller1990rflp, rick1982potential}.
Wild tomato relatives have been shown to harbor resistance genes for more than 40 agronomically important diseases, and breeders are working to introgress them into cultivated tomato \cite{rick1995utilization}\.
\cite{hanson2000mapping} mapped the tomato yellow leaf curl virus resistance gene in L. hirsutum f. glabratum accession 'B 6013' (line H24) (from \cite{banerjee1990transfer}) to an introgression on chromosome 11, which originated from the wild relative L. hirsutum.
Similar resistance traits has also been shown to have been imparted from L. chilense, L. pimpinellifolium, and L. cheesmanii [[various papers cited page 19 \cite{hanson2000mapping}\).

\subsection*{common bean}

Common bean was domesticated from a wild common bean at least twice (once in the Lerma-Santiago Basin of Mesoamerica and once in the southern Andes) \cite{kwak2009putative}\.
The domesticated common bean progenitor species is P. vulgaris var. aborigineus, which is found across the breadth of this region \cite{debouck1988implications]\.
Wild common bean now occurs across the Americas, from Mexico to Argentina, inhabiting a broad range of environments.
Because common bean is generally self-pollinating (outcrossing at an estimated rate under 3%, \cite{ramalho2006}/), rates of gene flow between it and wild relatives are expected to be lower.
\cite{papa2003asymmetry}\ shows little gene flow from wild common bean to domesticated.
Though some hybrids between wild and domesticated common bean show heterosis, others (especially F1 hybrids) do not, producing smaller plants and smaller seeds \cite{gutierrez1985heterosis}\.
It may take careful, larger-scale breeding programs to exploit heterotic hybrids for agronomic gain \cite{paredes1995extensive}\.
In the meantime, farmers may generally be selecting against hybrids.
Most common bean cultivation occurs in South America on small- and mediumholder farms \cite{paredes1995extensive}.
Because F1 hybrids are intermediate in seed size and color, farmers may effectively identify and avoid hybrid stock.
\cite{papa2003asymmetry}\ found that the spread of wild common bean populations may be facilitated by adaptive genes from domesticated common bean landraces through stable weedy hybrid populations, but little field evidence exists to support adaptive introgression in the direction of wild bean to domesticated.
\cite{papa2003asymmetry}\ was unable to determine how much of the genetic variability of domesticated common bean landraces originated from weedy or wild types, but point to the allele for 'L' phaseolin as a possible example of an allele that has been introgressed from wild common beans into domesticated.
For example, genetic material from cultivated common bean has been introgressed into wild populations, leading to increased seed size, an adaptation that permitted their spread to higher elevations with cooler climates \cite{debouck1993genetic}\.
This may be due in part to the large population size of common beans in a cultivated field relative to the population size of wild common beans within pollen dispersal range (a distance that may be quite small, \cite{papa2003asymmetry}\).
These weedy intermediates are occasionally harvested along with cultivated common bean and consumed by subsistence farmers \cite{papa2003asymmetry}\.
Common leaf blight and white mold resistances have experimentally bred into P. vulgaris L. from scarlet runner bean P. coccineus \cite{park1987transfer, schwartz2006inheritance}\, as was resistance to weevils Zabrotes subfasciatus and Acanthoscelides obtectus from Mexican wild beans \cite{kornegay1991inheritance}\.
Between the Mesoamerican and Andean domesticated gene pools, there is little opportunity for admixture due to geographical isolation, though their respective ranges do meet in some places, such as Columbia \cite{gepts1986phaseolin}\.
\cite{paredes1995extensive}\ suggests that common bean germplasm in Chile may be the result of introgression between the Mesoamerican and Andean gene pools, and breathes hope into the prospect of adaptive introgression in common bean breeding.
(Actually, the above suggestion was confirmed by \cite{kwak2009structure]\, citing that as many as 70% of Chilean landraces may be hybrid in origin [[(and by hybrid, I'm pretty sure they mean hybrids between the two domesticated forms of common bean, so i'm not actually sure it "breathes hope" into natural adaptive introgressive hybridization.)]]
\cite{kwak2009structure}\ also found via STRUCTURE analysis that certain Andean wild common bean accessions were possibly the result of hybridizations with domesticated common bean.
Furthermore, \cite{papa2003asymmetry}\ asks the reader to consider the view that the domestication of common bean was not a single event in the past, but is rather an ongoing "dynamic process resulting from selection, hybridization, and reselection over many years".
phaseolin?

\subsection*{Sorghum}
Anthropological evidence points to a domestication event of sorghum ((Sorghum bicolor subsp. bicolor) some 5,000 or 6,000 years ago Ethiopian Sudan \cite{smith2000sorghum}\.
However, comparative isozyme analyses \cite{shechter1975comparative}\ have suggested a possible secondary domestication event (sorghum race kafir) in the southeastern savanna.
The progenitor was S. arundinaceum \cite{doggett1988sorghum, harlan1971toward}\.
The genus Sorghum contains 25 species, but these are split between five subgenera: Eu-Sorghum, Chaetosorghum, Heterosorghum, Para-Sorghum, and Stiposorghum, and it is Eu-Sorghum that includes all domesticated, weedy, and closely-related wild relatives \cite{USDAARS2007, garber1950cytotaxonomic}.

Members of Sorghum bicolor are interfertile, though outcrossing rates and fertility can be low due to ploidy incompatability \cite{doggett1988sorghum, arriola1996crop}\.

fertility rates can be low due to incompatability and ploidy 

Rates of outcrossing within sorghum range from 0-30% \cite{doggett1988sorghum}\.

Within Eu-Sorghum, there are currently five cultivated (bicolor, guinea, kafir, caudatum, and durra) and four wild (arundinaceum, virgatum, aethiopicum, and verticilliflorum)

There are currently five races of S. bicolor subsp. bicolor currently cultivated: bicolor, guinea, kafir, caudatum, and durra \cite{smith2000sorghum}\.
Domesticate-weedy-wild complexes are common when sorghum cultivars are grown in sympatry with wild members of the Eu-Sorghum subgenus \cite{de1978systematics, doggett1968disruptive, baker1972human}\.
Natural introgression between wild and domesticated sorghum has been documented in both directions \cite{kuhlman2006interspecific, aldrich1992restriction, aldrich1992patterns, doggett1988sorghum, baker1972migrations}\.
These hybrids express diminished fertility

Members of Sorghum bicolor have adapted to a broad range of altitudinal, precipitation, and temperature clines across Africa and around the world \cite{po1982sorghum}\.

Breeding efforts have focused on incorporating exotic germplasm (wild sorghums as well as relatives from other genera, \cite{de1976cytogenetics}\) for its adaptations to biotic and abiotic stresses \cite{reddy2006current, po1982sorghum, johnson1979breeding}\.
Resistance to greenbug has been introgressed into cultivated sorghum through modern breeding efforts \cite{johnson1979breeding}\.
\cite{po1982sorghum}\ states that natural introgression has been an important force in sorghum evolution

\subsection*{Other Notes}

\cite{zohary2012domestication} has Map 1, showing the domestication center of einkorn wheat, emmer wheat, barley, chickpea, flax, lentil, pea, and bitter vetch in the Fertile Crescent.
Most of the wild progenitors of founder crops (excluding barley and flax) have a distribution restricted to the Fertile Crescent \cite{zohary2012domestication}.
However, such exceptions as barley, flax, foxtail millet, and oat have wild progenitors which inhabit a wide range of habitats beyond the Fertile Crescent.
\cite{papa2003asymmetry}\ In order to persist, cultivated-weedy-wild breeding complexes require a conducive environment in which all three biological components can coexist in close proximity.
Relevant environmental factors include biotic, climatic, and anthropogenic pressures.
\cite{jarvis1999wild} Table 2 is an incredible resource, cited examples of farmer selection and/or use of introgressed types, and confirmation of introgression
https://books.google.com/books?hl=en&lr=&id=CvcbUopfa54C&oi=fnd&pg=PA7&dq=common+bean+domestication&ots=ggAF-EoWG8&sig=dI9wOyATpzAelxLf_cg-gs0J2gI#v=onepage&q=common%20bean%20domestication&f=false

\subsection*{non-crop systems}
From {label3}
Bacteria, malaria mosquitoes, blackflies, Darwin's finches, butterflies (each example has a cited source, not included in the bibliography of this outline).
Also, non-crop plants, like trees (oak, larch).
From {label15}
Helianthus, Iris
Mouse, Salamander
From {label21}
Iris, Helianthus, Cowania/Purshia
Dacus (now Bactrocera), Anopheles
Haemophilus influenza, Trypanosoma cruzi
From {label23}
165 proposed cases of introgression, 65 "deemed to be sufficiently documented" (plants)
From {label28}
Milkweeds
From {label29}
White clover




\end{document}
