\documentclass[11pt]{article}
\usepackage{color}
\usepackage{authblk}%allows footnote format for authors
\usepackage[letterpaper, margin=1in]{geometry} %package that allows changes in margins and header/footers
\usepackage{natbib}
\bibliographystyle{ieeetr}
\newcommand{\mbh}[1]{\textcolor{green}{ \emph{\scriptsize  #1}} } %creating command for Matt's comments
\newcommand{\lwang}[1]{\textcolor{red}{ \emph{\scriptsize  #1}} } %creating command for Li's comments

\title{Crop adaptation through gene flow}

\begin{document}

\maketitle

\section*{Outline}






\subsection*{Maize}


Maize (\emph{Zea mays} ssp. \emph{mays}) is an ideal model system for the study of adaptive introgresssion from wild relatives.
The wild relatives and progenitors of maize are collectively known as teosinte (\emph{Zea mays}).
In particular, \emph{Zea mays} ssp. \emph{mexicana} (\emph{mexicana} hereafter) and Mexican highland maize have become a well-studied pair (although ssp. \emph{parvaglumis} has also been shown to hybridize with maize \cite{wilkes1977hybridization}).
Introgression between \emph{mexicana} and Mexican highland maize has been reported based on evidence from both morphological data \cite{wilkes1977, lauter2004, doebley1984} and molecular analysis \cite{matsuoka2002, vanHeerwaarden2011, doebley1987, warburton2011, fukunaga2005}. [[Li's and my bib tags might be in conflict, e.g. wilkes1977 is likely my wilkes1977hybridization.]]
Mexican farmers have long been aware of maize-teosinte hybrids, and in some places, such as the Nobogame valley, farmers have reported observing desirable traits passed to their crops through the hybrid plants \cite{wilkes1977hybridization} \cite{lumholtz1902unknown} \cite{wilkes1970teosinte}.
John Doebley (\cite{doebley1987patterns}) found that north-central high-elevation Mexican landraces (Apachito, Arrocillo Amarillo, Azul, Celaya, Chaloqueno, Conico, Conico Norteno, Gordo, Harinoso de Ocho Occidentales)[[special characters]] show evidence of introgression of two alleles from \emph{mexicana}, and suspected that they might have fitness advantages for adapting to highland environmental conditions.
Population structure analyses by \cite{matsuoka2002single} estimate that \emph{mexicana} gene flow has contributed to the gene pools of the Cacahuacintle, Palomero de Jalisco, and Palomero Taloqueno landraces, but also into (non-landrace) Mexican maize at the same elevation.
 
However, \cite{hufford2013} was the first to reveal genomic evidence of adaptive introgression from \emph{mexicana} to Mexican highland maize.
The authors identified nine genomic regions, which showed evidence of introgression from \emph{mexicana} to maize in both the HAPMIX and the linkage model of STRUCTURE analyses with over seven sympatric population pairs among the total nine pairs sampled. 
Among the nine regions, three spans the centromeres of chromosomes 5, 6, and 10, and one is located in the inversion polymorphism on chromosome 4, suggesting a significant role of genome structures restricting recombination in adaptive introgression.
By further characterizing the nine introgression regions, it is found that most regions contain long tracts of zero diversity, enriched with QTL linked with anthocyanin content and leaf macrohairs \cite{lauter2004} and over-represented with the SNPs demonstrating high association with temperature seasonality.
Growth chamber experiments with maize populations with introgression from \emph{mexicana} on chromosome 4 (associated with QTL controlling pigment density and macrohairs) and 9 (overlapped with QTL for macrohairs) exhibited more macrohairs and greater pigmentation under the highland environmental settings than the populations with absence of introgression from \emph{mexicana}.





\subsection*{Potato}

The northern members of the polyphyletic \emph{S. brevicaule} wild potato complex have been identified as likely progenitors of \emph{Solanum tuberosum}, the modern potato \cite{correll1962potato}, but determination of a single progenitor species is unlikely, either because widespread gene flow in the complex will mask signs of this phylogeny or because \emph{s. tuberosum} has a polyphyletic origin.
Also, these northern members of the \emph{S. breviaule} complex are not clearly defined, and may in fact be one singular species (in which case the species name would be \emph{S. bukasovii} \cite{spooner2005single}.

Although potatoes are usually propogated clonally, farmers also promote sexual hybridization at times to improve disease resistance and develop new cultivars \cite{quiros1992increase}\.
Farmers continue to grow potatoes in close proximity to wild relatives, resulting in domesticate-weedy-wild hybrid complexes which promote introgressive hybridization \cite{rabinowitz1990high, johns1987relationships, linder1987diversity}\.
These complexes, combined with a diverse range of biotic and environmental selective pressures and local farming practices (human-mediated migration, isolated farmsteads in fertile valleys, clonal propogation, and intentional maintenance of a variety of landraces), have fostered expansion of genetic diversity within potatoes subsequent to domestication \cite{brush1995potato}\.
However, as farmers tend to abandon fields after being used for potato cultivation, it is less likely that hybrids have an opportunity to form stable populations for maintained introgressive gene flow into the domesticated potato gene pool \cite{brush1995potato}\.

Introgressive hybridization is widespread in potatoes \cite{grun1990evolution}\.
The various cultivars of Andean potatoes are interfertile, forming one large plastic gene pool \cite{quiros1992increase}\ (fueling the assertion that these potatoes comprise a single species).
Andean potatoes exhibit high ecological versatility, due in part either to alleleic diversity in polyploids or introgression of desirable alleles from wild relatives in diploids \cite{zimmerer1998ecogeography}\.
Cultivated diploid potatoes have been shown to be introgressed with genes from \emph{S. sparsipilum} \cite{rabinowitz1990high}\ and \emph{S. megistacrolobum} \cite{johns1987relationships, huaman1980solanum}.
Drought- and frost-resistance genes introgressed from wild relatives facilitated the spread of cultivated potatoes into the central Andean Altiplano \cite{johns1986ongoing, hawkes1962origin, schmiediche1980breeding}\.
Resistance genes have been experimentally introgressed from wild poptato relatives (\emph{S. tuberosum subsp. andigena}, \cite{van1999tight}, \emph{S. bulbocastanum}, \cite{van2003ancient}) into cultivated potato, although this process is hindered by compatability issues including ploidy levels and Endosperm Balance Number \cite{johnston1980significance}.






\subsection*{Sunflower}

Domesticated sunflower \emph{Helianthus annuus} lives in sympatry with wild relatives like \emph{H. petiolaris} and \emph{H. bolanderi} and forms stable hybrid populations \cite{schwarzbach2002likely, rieseberg1988molecular, welch2002patterns}\.
Wild sunflowers are locally adapted, and weedy hybrid populations share these adaptations \cite{kane2008genetics}\.
Domesticated sunflower shows signs of persistent introgressive hybridization with \emph{H. petiolaris} with evidence of positive selection driving some of the genetic differentiation between the two species \cite{yatabe2007rampant}\.
Domesticated sunflower adapted to the environment in Texas by hybridizing with cucumberleaf sunflower (\emph{H. debilis ssp. cucumerifolius}), gaining advantageous alleles (these hybrids are now called \emph{H. annuus ssp. texanus}, \cite{kim1999genetic, heiser1951hybridization, rieseberg1999hybrid, rieseberg1990helianthus}.

There are several downy mildew resistance genes in \emph{Helianthus}, and each imparts resistance to one or more races of \emph{Plasmopara halstedii}, one of the most agronomically important diseases in sunflower cultivation \cite{cohen1973factors}\.
Some of these downy mildew resistence genes were found in wild relatives (including \emph{H. argophyllus}, \emph{H. tuberosus}, and \emph{H. praecox}) and have been successfully bred into modern \emph{H. annuus} \cite{miller1991inheritance}\.
PlArg, an allele found in wild silverleaf sunflowers (\emph{H. argophyllus}, inbred line Arg1575-2), confers resistance to all known (20 or more) races of downey mildew \cite{dussle2004pl}\, while others alleles (Pl1-Pl11) are effective for one or more types \cite{rahim2002inheritance}\.
Silverleaf sunflower has also been the focus of drought resistance breeding efforts \cite{saucaÌ†2010introgression}\ and \emph{Phomopsis} resistance breeding efforts \cite{besnard1997specifying}.





\subsection*{Sorghum}
The genus Sorghum contains 25 species, but these are split between five subgenera: Eu-Sorghum, Chaetosorghum, Heterosorghum, Para-Sorghum, and Stiposorghum, and it is Eu-Sorghum that includes all domesticated, weedy, and closely-related wild relatives \cite{USDAARS2007, garber1950cytotaxonomic}.
Within the sub-genus Eu-Sorghum, there are currently nine races, five cultivated (bicolor, guinea, kafir, caudatum, and durra) and four wild (arundinaceum, virgatum, aethiopicum, and verticilliflorum) \cite{smith2000sorghum}\.
Members of \emph{Sorghum bicolor} are interfertile, though outcrossing rates and fertility can be low due to ploidy incompatability \cite{doggett1988sorghum, arriola1996crop}.

\cite{po1982sorghum}\ states that natural introgression has been an important force in sorghum evolution.
Domesticate-weedy-wild complexes are common when sorghum cultivars are grown in sympatry with wild members of the Eu-Sorghum subgenus \cite{de1978systematics, doggett1968disruptive, baker1972human}\, but these hybrids exhibit diminished fertility.
Rates of outcrossing within sorghum range from 0-30% \cite{doggett1988sorghum}\, and Natural introgression between wild and domesticated sorghum has been documented in both directions \cite{kuhlman2006interspecific, aldrich1992restriction, aldrich1992patterns, doggett1988sorghum, baker1972migrations}\.
Resistance to greenbug has been introgressed into cultivated sorghum through modern breeding efforts \cite{johnson1979breeding}.




\subsection*{Rice}

The two main cultigens of rice are Asian rice (\emph{Oryza sativa}) and African rice (\emph{O. glaberrima}), and they are easily distinguish by ligule length, number of secondary panicle branches, panicle axis thickness, and differences in life cycles \cite{oka2012origin}.
Within \emph{O. sativa}, there are two main subspecies; Indica (with subpopulations indica and aus) and Japonica (with subpopulations temperate japonica, tropical japonica known as javanica, and aromatic) \cite{chang2003origin, glaszmann1987isozymes, ricepedia}.
Indica and Japonica are likely the result of independent domestications from separate \emph{O. rufipogon} populations in India/Indochina and southern China, respectively \cite{londo2006phylogeography}\.
These subspecies display adaptations to the environmental coniditions corresponding to the differentiated geographical locations they inhabit \cite{khush2003classifying}.

Both Asian and African rice naturally hybridize with other domesticated subspecies and with wild relatives (of which there are about twenty \cite{ricepedia}), and introgression is common \cite{oka2012origin, second1982origin, zhao2010genomic}\.
The greater genetic diversity within \emph{O. sativa} is likely due to introgression with wild relatives both during domestication and upon the dispersal of \emph{O. sativa} into new environments and sympatry with new relatives \cite{second1982origin}.
Resistance alleles to several pests (grassy stunt virus, bacterial blight, brown planthopper, blast) have been introgressed from wild relatives into \emph{O. sativa} by researchers \cite{brar1997alien, khush1974inheritance}.
Beyond investigative endeavours, gene flow from wild relatives has been used to produce agronomic rice varieties.
Yatsen No. 1, for example, showed resistance to pests and diseases and adapted well to environmental conditions \cite{ting1933wild}\.
Several lines were derived from Yatsen No. 1, and went on to be utilized extensively in parts of China.





\subsection*{tomato}

And yet, \cite{bai2007domestication} claim that "[m]ost likely, no exhange of genetic information with the wild germplasm took place until 1940."
It seems more likely that such genetic exchange would simply have been unintentional, rather than strictly nonexistent.
Since the work of Charles Rick in 1940s and 50s \cite{rick1953novel}\, breeding efforts have focused on incorporating and utilizing the relatively diverse genetic germplasm of wild tomato relatives \cite{rick1988tomato, miller1990rflp, rick1982potential}\.
Wild tomato relatives have been shown to harbor resistance genes for more than 40 agronomically important diseases, and breeders are working to introgress them into cultivated tomato \cite{rick1995utilization}\.
\cite{rick1958role} summarizes morphological evidence of natural introgressive hybridization with wild relatives in South America, despite cultivated tomato's proclivity to self-pollinate.
Also, \cite{hanson2000mapping} mapped a tomato yellow leaf curl virus resistance gene in an accession of \emph{L. hirsutum f. glabratum} \cite{banerjee1990transfer} to an introgression on chromosome 11, which originated from the wild relative \emph{L. hirsutum}.
Similar resistance traits has also been shown to have been imparted from \emph{L. chilense}, \emph{L. pimpinellifolium}, and \emph{L. cheesmanii} \cite{hanson2000mapping}.
[[Dig deeper into these sources - evidence?  and naturally?]]





\subsection*{wheat}

The convoluted domestication of wheat is marked by many instances of hybridization between morphologically-distinct species within a restrained geographical range.
Throughout wheat domestication, there has been gene flow between species and between wild and domesticated wheats.
Emmer wheat, for example, has high genetic diversity, likely due to gene flow from wild emmer \cite{luo2007structure, dvorak2006molecular}\.
The a, b, c, and f alleles of mildew resistance gene Pm3 may have been introgressed into wheat from wild emmer wheat shortly after domestication (although the e, d, and g alleles were probably formed by de novo mutations) \cite{TPJ:TPJ2772}\.
Furthermore, wild wheat relatives are can be found in a much broader swath than the domesication center in the Fertile Crescent \cite{CWR}\.





\subsection*{Cotton}

Gossypium hirsutum is the most important and most agronomically grown cotton today.
G. hirsutum is from Mexico.
There are many species in the genus, from around the world, and many have been cultivated.

percy1990allozyme:
G. barbadense is grown commercially in S. America, where it hybridizes with G. hirsutum as well

wendel1989genetic:
bidirectinal gene flow between diploid species G. arboreum and G. herbaceum
independent domestication events of G. arboreum and G. herbaceum

The old-world A genome became part of the new-word genome in the polyploidization event; talk about this?

abdalla2001genetic:
Lots of interbreeding between cultivated lines of cotton, but what about wild relatives?

Cotton readily tolerates interspecific crossing.
This leads to a reticulated phylogeny.







\subsection*{Common Bean}

Because common bean is generally self-pollinating (outcrossing at an estimated rate under 3%, \cite{ramalho2006}\), rates of gene flow between it and wild relatives are expected to be lower.
These expectations are supported by evidence provided by \cite{papa2003asymmetry}\.
However, hybrid populations can still be found in sympatry with wild and domesticated common bean, forming domesticate-hybrid-weedy complexes [[perhaps use kwak2009structure?  would have to check again]].

Some evidence for introgression exists in the direction of domesitcated to wild common bean, leading to increased seed size, an adaptation that permitted their spread to higher elevations with cooler climates \cite{debouck1993genetic}\.
Also, \cite{kwak2009structure} found via STRUCTURE analysis that certain Andean wild common bean accessions were possibly the result of hybridizations with domesticated common bean.

Though limited, there is also some evidence of natural introgression of wild genes into domesticated common bean.
\cite{papa2003asymmetry}\ was unable to determine how much of the genetic variability of domesticated common bean landraces originated from weedy or wild types, but point to the allele for 'L' phaseolin as a possible example of an allele that has been introgressed from wild common beans into domesticated.
Experimentally, common leaf blight and white mold resistances have been bred into common bean from scarlet runner bean (\emph{P. coccineus}) \cite{park1987transfer, schwartz2006inheritance}\, as was resistance to weevils (\emph{Zabrotes subfasciatus} and \emph{Acanthoscelides obtectus}) from Mexican wild bean species \cite{kornegay1991inheritance}\, but it may take careful, larger-scale breeding programs to exploit heterotic hybrids for agronomic gain \cite{paredes1995extensive}.





[[Olive, Sugarcane, Soybeans, Barley show similar patterns (ie promising conditions but no direct evidence).]]

Olive

Olives (Olea europaea subsp. europaea var. sativa) were domesticated primarily in the northeastern Levant and then dispersed west along the length of the Mediterranean range of wild (var. oleaster) olives \cite{besnard2013complex}\.
The prevalence of wild olives (var. oleaster) has since been diminished.

Morphological similarities indicate that wild, weedy, and cultivated olives form an interfertile species complex that sprawls across the Mediterranean, and that outbreeding with these wild and weedy relatives has contributed to cultivated olive's genetic diversity, but it has not been specifically shown that locally-adapted genes have been introgressed into the domesticated olive \cite{zohary1975beginnings}\.

Olives have been propogated clonally for much of their history, limiting gene flow, but given the chance, cultivated and wild olives are fully interfertile [[green and wickens 1989]].
Feral cultivated/wild hybrid oleasters are evidence of such outcrossing events.

Direct evidence of adaptive wild-to-crop introgressive hybridization is at present absent, however.




Sugarcane

From \cite{ellstrand1999gene}\:
33 - daniels1987taxonomy
 "genetically based traits" evidence for hybrid origin of "many cultivars"
 "chromosomal evidence" supports "spontaneous intergeneric hybridization" origin of "particular accessions of wild sugarcane relatives"
 Erianthus maximus might be a hybrid of S. officinarium and Miscanthus (a wild relative genus), based on morphological and chromosomal evidence.
104 - roach1995sugar
 "spontaneous hybridization between cultivated and wild Saccharum in Australasia and islands of the Indian Ocean" in origin of "many cultivars"
115 - sobral1994phylogenetic
117 - stevenson1965genetics
 "certain wild canes from Java and Mauritius are hybrids between" wild (S. spontaneum) and cultivated (S. officinarum)
 
 from roach1995sugar:
 Spontaneous hybridization in Australasia with wild relatives.
 No evidence yet of adaptive introgression, though.






/subsection*{Soybean}
Soybeans (Glycine max) were domesicated in southern China \cite{guo2010single}\.
Soybeans (Glycine max) primarily self-pollinate, outcrossing at a rate of 1-8% \cite{ray2003soybean}\.
Wild soybeans outcross at a rate between 9.3 and 19% \cite{fujita1997extent}\.
Soybeans readily cross with their wild progenitor, G. soja \cite{singh1988genomic}\.
Plants of intermediate morphology appear around fields when S. soja is also present, indicating spontaneous hybridization \cite{kwon1972studies}\.
The accession "G. gracilis" shows allelic contribution from both G. max and G. soja \cite{keim1989restriction}\.
QTLs for adaptive traits (pest resistance, disease resistance, seed quality, etc.) have been identified and described [[see the highlighted intro paragraph in concibido2003introgression]]






\subsection*{barley}

Domesticated and wild barleys belong to the same species, \emph{Hordeum vulgare}, and are biologically capable of producing viable offpsring via hybridizaion \cite{von1995ecographical}.

Variety agriocrithon (\emph{H. vulgare ssp. vulgare f. agriocrithon}) is genetically diverse, and is found throughout much of the range of Barley.

The barley domestication process has reduced the number of alleles in the domesticate to only 40% of that found in wild barley, though there remains a great deal of phenotypic diversity among the wild barleys \cite{ellis2000wild}\.
Wild-domesticate breeding experiments have shown that wild barleys have alleles for several important agronomic phenotypes, including brittleness, flowering time, plant height, lodging, and yield,  \cite{von2006ab, handley1994chromosome}\.
Although the conditions of barley domestication would seem to allow (if not promote) natural adaptive introgression between barley and its wild relatives, there is little evidence of this genetic interaction at present.
Owing to this sparcity of evidence is the low rate of outcrossing of \emph{spontaneum}, estimated by \cite{brown1978outcrossing} to be 1.6% in Israeli populations.
There has been little genetic investigation into spontaneous barley/\emph{spontaneum} hybrids \cite{ellstrand2003dangerous}\.
Barley/\emph{spontaneum} hybrids are fertile, and morphologically intermediate (putatively hybrid) barleys are found when wild and domesticated barleys are grown in sympatry, but hybrids of other wild relatives generally exhibit greatly diminished fertility \cite{ellstrand2003dangerous, harlan1995living}\.
Even when the two are not grown immediately adjacent to one another, introgression from wild to domesticate has been shown to happen over distances of more than a kilometer \cite{hillman2001new}.



\bibliography{bib_gj.bib}
\end{document}
