\documentclass[11pt]{article}
\usepackage{cite}
\usepackage{color}
\usepackage{pgf}
\usepackage[normalem]{ulem}
\usepackage{multibib}

\title{Crop adaptation with gene flow from wild relatives}

\author[1]{author 1}%author information
\author[1]{author 2}
\author[1, 4]{author 3}
\affil[1]{Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa, USA}
\affil[4]{Corresponding Author: Matthew B. Hufford; 339A Bessey Hall, Iowa State University, Ames, IA, USA; phone: 1-515-294-8511; email: mhufford@iastate.edu}
\date{}

\begin{document}

\maketitle

\section{Perspectives and key issues}
\begin{itemize}
%\item ways to detect gene flow 
%compromise between sample size of the source population and the SNP density across the genome
%\item ways to detect selection
%The methods to detect both gene flow and selection are summarized in Racimo et al. (2015) (archaic adaptive introgression in humans)
\item What is the proportion of immigrants showing the signatures of adaptation across the genome
\item The extent to which adapted immigrants are physically linked on chromosomes and reside within structural features restricting recombination, such as inversions
\item the importance of genome structures on local adaptation
An antagonism exists between selection for combination of locally adapted genes, and migration and recombination tending to break them down.
Therefore, genome structures, such as inversion and centromeres, which contrains the recombination, can boost the effectiveness of selection by extending the linkage disequilibrium.
\end{itemize}


\section{theory of adaptative introgression}

\section{Empirical Data}
\subsection{maize}
Maize is a classical model in crop species to study the issue of adaptive introgresssion from wild relatives, as it is sympatric with a lot of its wild relatives with a range of different divergence time \cite{hufford2013}.
\emph{Zea mays} ssp. \emph{mexicana} (referred to \emph{mexicana} hereafter) and Mexican highland maize is a well-studied pair.
Introgression between \emph{mexicana} and Mexican highland maize has been reported based on evidence from both morphological data \cite {wilkes1977, lauter2004, doebley1984} and molecular analysis \cite{matsuoka2002, vanHeerwaarden2011, doebley1987, warburton2011, fukunaga2005}.
However, \citep{hufford2013} for the first time revealed the evidence of adaptive introgression from \emph{mexicana} to Mexican highland maize.
The authors identified nine genomic regions, which showed evidence of introgression from \emph{mexicana} to maize in both the HAMPMIX and the linkage model of STRUCTURE analyses with over seven sympatric population pairs among the total nine pairs sampled. 
Among the nine regions, three spans the centromeres of chromosomes 5, 6 and 10 and one is located in the inversion polymorphism on chromosome 4, suggesting a significant role of genome structures restricting recombination in adaptive introgression.
By further characterizing the nine introgression regions, it is found that most regions contain long tracts of zero diversity, enriched with QTL linked with anthocyanin content and leaf macrohairs \cite{lauter2004} and over-represented with the SNPs demonstrating high association with temperature seasonality.
It was also revealed that in the growth chamber experiment maize populations with introgression from \emph{mexicana} on chromosome 4 (associated with QTL controlling pigment density and macrohairs) and 9 (overlapped with QTL for macrohairs) exhibited more macrohairs and greater pigmentation under the highland environmental settings than the populations with absence of introgression from \emph{mexicana}.
To put it together, the introgressed alleles/haplotypes from \emph{mexicana} to maize conferred adaptation to highland habitats when maize migrated from Mexican lowlands.
%Finally, the contribution of \emph{mexicana} genetic resource to a broader maize panel was evaluated by analyzing patterns of identity by state (IBS) between maize and \emph{mexicana} as well as between maize and \emph{Zea mays} ssp. \emph {pulviglumis} (referred to \emph {pulviglumis} hereafter).

Even though introgression from \emph{mexicana} to Mexican highland maize has been highly supported \cite{hufford2013}, it is yet unknown whether and to what extent such introgression could be found in other highland maize populations, which are allopatric to \emph{mexicana}. 
\citep{Takuno2015} found little empirical and theoretical support for parallel highland adaptation in the Mexican and South American highland regions, which was explained partly owing to the difference in the potential for adaptive introgression from wild relatives.
Adaptive SNPs in Mexican highland population were more likely located in the introgressed regions than those in South American highland populations.
Furthermore, the adaptive SNPs in Mexican highland population were more likely also showing signatures of local adaptation in \emph{mexicana} and \emph {pulviglumis} populations than those South American highland population.
To sum up, the adaptive introgression from wild relatives may play a significant role in patterning the genetic differences of maize highland populations.
A recent study \cite{Wang2015manuscript} revealed that gene flow exist from \emph{mexicana} not only to Mexican highland maize, but also to Guatamalan highland maize, as well as to some individuals in Southwestern US Highland and even Andes, suggesting different extent contributions \emph{mexicana} made to multiple maize highland landrace populations depending on the distance away from the distribution of \emph{mexicana}. 
However, whether the introgression from \emph{mexicana} to those highland populations contributes to the highland adaptation is awaiting for future research. 
 
\subsection{barley}
\subsection{wheat}
\subsection{sunflower}

\section{Concluding remarks and future directions}


\bibliography{Reference}
%\bibliographystyle{...}
%\onecolumn
%\input{supportingInfo} 

% taxa in selection/migration equilibrium
%transition from individual gene to whole-genome understanding
% extent of local adaptation to environmental factors, as well as patterns of gene flow
% migrant alleles or migrant haplotypes





\end{document}
