\documentclass[11pt]{article}
\usepackage{color}
\usepackage{authblk}%allows footnote format for authors
\usepackage[letterpaper, margin=1in]{geometry} %package that allows changes in margins and header/footers
\usepackage[numbers,sort]{natbib}
\usepackage{amsmath}
\usepackage{rotating}
\usepackage{adjustbox}
\usepackage[english]{babel}
\usepackage{colortbl}
\usepackage{booktabs}
\usepackage[x11names,dvipsnames,table]{xcolor}
\bibliographystyle{ieeetr}
\newcommand{\mbh}[1]{\textcolor{orange}{ \emph{\scriptsize  #1}} } %creating command for Matt's comments
\newcommand{\lwang}[1]{\textcolor{red}{ \emph{\scriptsize  #1}} } %creating command for Li's comments
\newcommand{\gmj}[1]{\textcolor{blue}{ \emph{\scriptsize  #1}} } %creating command for Garrett's comments

\title{Review: Redefining Domestication: Adaptive Introgression during Crop Expansion}

\author[1]{Authors: Garrett M. Janzen}%author information
\author[1]{Li Wang}
\author[1,*]{Matthew B. Hufford}
\affil[1]{Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa, USA}
\affil[*]{Correspondence: mhufford@iastate.edu (M.B. Hufford)}
\date{}

\begin{document}



\maketitle

%Short Research reviews should be in the range 3500–4000 words, with up to 40 references and 6 figures/tables.


The process of domestication was once thought to be rapid and geographically constrained, with crops originating from a wild progenitor within one or more geographically defined centers followed by expansion to the modern-day extent of cultivation. \mbh{will need citation for this}
\gmj{ the paper "Domestication rates in wild-type wheats and
barley under primitive cultivation" states that domestication could have taken place in as little as 20 or 30 years.}
However, archaeological and genetic evidence are beginning to reveal that, in many cases, domestication has been temporally protracted and more diffuse \cite{brown2009complex}. \mbh{citations here to Li's paper, Purugganan rice paper, Brandon Gaut's recent grape paper} \gmj{the rice paper you mentioned, I think, is "ARCHAEOLOGICAL DATA REVEAL SLOWRATES OF EVOLUTION DURING PLANTDOMESTICATION".  This paper suggests that the rate of phenotypic change in domestication is slower than expected, nearly that of natural selection in wild populations.  Also, see the first paragraph of the Discussion for a list of older papers supporting rapid domestication.}
This new conception of domestication emphasizes the role of beneficial gene flow (\emph{i.e.}, adaptive introgression) from locally adapted wild relatives during crop expansion after initial domestication.


Adaptive introgression has three components: hybridization between two genomes, backcrossing to one of the parents, and selection on different recombinant genotypes with progressively diminished linkage drag \cite{barton2001role, Feuillet200824}.
In domesticated species, adaptive introgression would consist of crop/wild hybrids backcrossing to a crop, retention and increase in frequency of adaptive wild haplotypes in the crop, and removal of undesirable wild background.
To date, literature on crop-wild gene flow has focused on the risk of introgression of transgenes from domesticated crops into wild relatives (for a review, \cite{stewart2003transgene}) and on modern plant breeding efforts to introgress desired traits from wild relatives (for a review, \cite{zamir2001improving, tanksley1997seed, hajjar2007use}). \mbh{cite one paper here...the best, most recent and comprehensive review}
The history of natural introgression of wild alleles into domesticated crops over evolutionary timescales has received considerably less attention.
However, recent tools and methods have been employed to detect genome-wide patterns of introgression, granting new insights into the prevalence of adaptive introgression in crop histories.
Emerging results suggest a need for reevaluation of the existing domestication paradigm.


In this review, we will: 1) briefly describe recently developed methods for detecting adaptive introgression and provide a summary of their application for detecting crop-wild introgression, 2) review evidence supporting the hypothesis that wild-to-crop introgression has conferred local adaptation, 3) consider how the prevalence of this introgression alters traditional concepts of domestication, and 4) describe future advances in both basic and applied genetics that can be made through the study of introgression in agroecosystems.











\section*{Introgression methods and their application}

The decreasing cost of genome-wide resequencing and availability of reduced-representation genotyping (\emph{e.g.}, GBS and RAD-Seq), combined with new analytical methods, has facilitated comprehensive study of introgression across a number of species (\textbf{Table 1}).
High-density marker data can be used with haplotype-based and other methods to assign specific genomic regions to a taxon of origin and identify introgression across taxa \cite{Martin2015,Price2009,Lawson2012,pease2015,rosenzweig2016,geneva2015}.
The methods reviewed here do not include those marginally estimating introgression\slash migration rate as a component of demographic history (\emph{e.g.}, Approximate Bayesian Computation (ABC) \cite{beaumont2002}, diffusion approximations for demographic inference ($\delta a\delta i$) \cite{gutenkunst2009}, isolation with migration models \cite{hey2004}, and the multiple sequentially Markovian coalescent (MSMC) \cite{schiffels2014}). 
Rather, we focus on methods that explicitly identify introgressed genomic segments based on patterns of nucleotide/haplotype diversity and differentiation and on phylogenetic relationships.
%We focus here on analytical prospects and limitations and introduce only a few representatives of each category.

First, introgressed segments are expected to show low differentiation from their source population.
The $F_{st}$ and $d_{XY}$ statistics and their derivates including $G_{min}$ \cite{geneva2015} and $RND_{min}$\cite{rosenzweig2016} gauge differentiation. 
The former two statistics are insensitive to rare migrants and therefore lack power to detect recent introgression, while the latter two overcome this limitation.
Additionally, $RND_{min}$ accounts for variable mutation rate, which is detected based on branch length to an outgroup. 

\mbh{abrupt transition here to Gmin equation...can you smooth this into the text?}
 
 \begin{equation}
    G_{min} = \frac{d_{min}}{d_{XY}}
 \end{equation}
 where $d_{min}$ is the minimum sequence distance between haplotypes in species X and Y.
 
 \begin{equation}
 	RND_{min} = \frac{d_{min}}{d_{out}}
 \end{equation}
where $d_{out}$ equals $(d_{XO} + d_{YO})/2$, the average sequence distance between each species and the outgroup ($O$).
 
These statistics have recently been further developed by adding differentiation between both non-admixed ($A$) and admixed populations ($B$) and a source population ($C$) \cite{racimo2016}. 
For example, the $U_{A,B,C(w,x,y)}$ statistic summarizes number of sites, where an allele at frequency $y$ in the source population ($C$) has a frequency higher than $x$ in the admixed population ($B$) and lower than $w$ in the non-admixed population ($C$).
A similar statistic, $Q95_{A,B,C(w,y)}$, sets a hard cutoff at the $95^{th}$ percentile of allele frequencies in the admixed population (B) \cite{racimo2016}.
\mbh{Did I get this right, Li?} 
Further modifications have allowed specification of two source populations (see details in \cite{racimo2016}).
 
Second, local ancestry deconvolution (also known as chromosome painting) assigns genomic regions to various source populations \cite{schraiber2015}. \mbh{How?  How do these methods work?  I know at least some are based on Hidden Markov Models...need a sentence or two on the nuts and bolts}
Such methods typically require phased haplotypes as input (but see \cite{churchhouse2013}).


Third, the ABBA-BABA statistic (also known as the D-statistic) and its derivatives are widely applied to introgression detection and make inferences based on genomic patterns of shared derived variants between populations or species in a phylogenetic context. \mbh{need sentence here explaining what D-statistic does} 
Elaborations of the D-statistic include $\hat{f_{d}}$ \cite{Martin2015} and the five-taxon D-statistic \cite{pease2015}. 
The former is quite similar to the D-statistic but uses allele frequencies from each population/species, and the latter detects introgression based on the localized phylogenetic pattern. \mbh{need to clarify here how this is different than $\hat{f_{d}}$}
%Their strength in identifying specific regions enables one to explore the relationships between introgression and recombination rate/gene density/distribution of deleterious alleles and to better understand the genomic regions penetrable to foreign gene flow. 

Fourth, approaches that assign genome-wide genetic ancestry are based on the popular software STRUCTURE \cite{pritchard2000}. 
For example, fineSTRUCTURE \cite{Lawson2012} calculates the probability that genomic regions belong to genetic groups by \mbh{need details on method here}. 
GLOBETROTTER \cite{hellenthal2014} is a further development of fineSTRUCTURE that allows for identification of an unsampled source population and for dating of admixture events.

Application of these approaches across a number of plant and animal species suggests introgression can play an adaptive role. For example, introgression from ancient hominins (\emph{e.g.}, Neanderthals and Denisovans) to humans has been detected at loci controlling skin pigmentation, defense against pathogens, and toleration of high altitude (reviewed in \cite{Racimo2015}), introgression has conferred M\"{u}llerian mimicry \mbh{I would explain a bit more here...wing coloration loci, protects against predation...} across butterfly species \cite{Heliconius2012} introgression has spread insecticide resistance across mosquito species \cite{Norris2015}, and introgression across \emph{Mimulus} (\emph{i.e.}, monkeyflower) species has resulted in adaptation to pollinator preference and contributed to speciation \cite{Stankowski2015}.


\noindent{\textbf{Table 1:} List and brief description of recently developed methods and examples of empirical studies employing these methods.}

\noindent{\textbf{Figure 1}: Wing coloration patterns in \emph{Heliconius} and evidence for introgression across species based on Patterson's \emph{D}-statistic; adapted from \cite{Heliconius2012}.}


%\lwang{In this part, maybe better to incorporate this reference, Detection and Polarization of Introgression in a Five-Taxon Phylogeny; and a new method published this year RND\_min Powerful methods for detecting introgressed regions from population genomic data; I donot know if we should also include G\_min, that is the statistic we tried that didnot work well for us: A new method to scan genomes for introgression in a secondary contact model. I put the three in the bib file.}





\section*{Crop adaptation through introgression}


Over the last few years, several high-profile publications based on genome-wide data have documented introgression between crops and their wild relatives outside putative domestication centers.
Recent empirical studies have revealed that introgression has occurred in many of the world's most important crops (\textbf{Table 2}).






\begin{enumerate}
\item{Maize:}






The relationship between maize (\emph{Zea mays} ssp. \emph{mays}) and the teosinte \emph{Zea mays} ssp. \emph{mexicana} (hereafter referred to as \emph{mexicana}) offers a prime case study of adaptive wild-to-crop introgression.
Maize was domesticated from (\emph{Zea mays} ssp. \emph{parviglumis}) approximately 9,000 years ago in the lowlands of the Balsas River Valley in Mexico \cite{matsuoka2002single}.
From this domestication center, maize spread into the highlands of the Mexican Central Plateau, where it came into sympatry with wild \emph{mexicana}.
Introgression from \emph{mexicana} to maize in the highlands of Mexico has been reported based on evidence from both morphological data \cite {wilkes1977, lauter2004, doebley1984} and molecular analyses \cite{matsuoka2002, vanHeerwaarden2011, doebley1987, warburton2011, fukunaga2005}.
However, \citep{hufford2013} first localized \emph{mexicana} introgression to chromosomal regions and provided evidence that it was likely adaptive.
%The following is not fully clear.  Cut or revise.
The authors identified nine genomic regions in several maize populations which showed evidence of\emph{mexicana} introgression based on local ancestry deconvolution using HAPMIX and results from the linkage model of STRUCTURE.
These introgressed segments showed low diversity and overlapped QTL that had previously been found to control anthocyanin content and leaf macrohairs \cite{lauter2004}, traits known to be important in adaptation to high elevation.
In a growth chamber experiment, the authors demonstrated that maize populations with \emph{mexicana} introgression showed greater plant height (a proxy for fitness) under highland environmental settings than populations that lacked introgression.
Height differences were not detected under lowland conditions.
%Among the nine regions, three span the centromeres of chromosomes 5, 6, and 10, and one is located in the inversion polymorphism on chromosome 4, suggesting a significant role of genome structures restricting recombination in adaptive introgression.
%In summary, the introgressed alleles/haplotypes from \emph{mexicana} to maize conferred adaptation to highland habitats when maize migrated from Mexican lowlands.


%Even though introgression from \emph{mexicana} to Mexican highland maize has been highly supported, it is yet unknown whether and to what extent such introgression could be found in other highland maize populations which are allopatric to \emph{mexicana}. 
%A recent study \cite{Takuno2015} found little empirical or theoretical support for parallel highland adaptation in the Mexican and South American highland regions, which was explained partly by the difference in potential for adaptive introgression from wild relatives.
%Adaptive SNPs in Mexican highland population were more likely located in the introgressed regions than those in South American highland populations.
%Furthermore, the adaptive SNPs in the Mexican highland population were more likely also showing signatures of local adaptation in \emph{mexicana} and \emph {pulviglumis} populations than those South American highland population.
%For these reasons, adaptive introgression from wild relatives may play a significant role in patterning the genetic differences of maize highland populations.

Populations of \emph{mexicana} cannot be found outside of Mexico, yet maize has colonized and adapted to high elevation in a number of additional regions.
%Several questions regarding \emph{mexicana} introgression yet remain.
%Has the introgression from \emph{mexicana} spread to multiple highland populations? 
%How do highland-adaptation traits differ among populations with and without introgression from \emph{mexicana}?
A recent study \cite{Wang2015manuscript} employed the ABBA-BABA and $\hat{f_{d}}$ statistics to evaluate whether maize with \emph{mexicana} introgression was transferred to other highland regions or whether highland adaptation was obtained \emph{de novo} outside of Mexico (Figure \ref{fig:fd}).
%On chromosome 4 (Figure \ref{fig:fd}), Mexican highland (MexHigh) and Guatamalan highland (GuaHigh) exhibited strong evidence of introgression from \emph{mexicana}, and the peaks of distribution corresponded to the region identified in Hufford et al. (2013) \citep{hufford2013}.
%The signal of introgression is absent for the other three populations.
%On chromosome 5, the signals of introgression (the peak region of the distribution) are present in MexHigh, GuaHigh and Southwestern US Highland (SW\_US), but not the other two.
%More details on the other chromosomes can be found in \cite{Wang2015manuscript}.
Overall, analyses revealed that maize landraces with \emph{mexicana} introgression were transferred to nearby high elevation regions in Guatemala and the southwestern United States, but that more distant high elevation regions (\emph{e.g.,} the Andes) showed no \emph{mexicana} ancestry. \mbh{on the fence about including deleterious allele results from introgressed regions}
%The Andean maize, the population totally isolated from the occurrence of any teosinte species, underwent the severest historical bottleneck, as a population in the front wave of the serial founder effects. 
%The stronger genetic drift introduced by the smaller historical population size increased homozygosity but reduced the heterozygosity of the population, and it also extended the length of homozygosity.
%The high frequency of deleterious alleles caused by stronger genetic drift in the Andes population, together with the reduced efficiency of selection against deleterious sites, contributes to the observed higher mutation load in the well-isolated maize population.
%Although it is clear that the absence of introgression from wild relatives provided fewer genetic resources for highland adaptation (making the highland adaptation in the Andes unique), it is yet unknown whether or not being out of reach of wild relatives is a reason for reduced fitness in the Andes population.
%Furthermore, the question of whether convergent evolution occurs between populations with and without introgression from wild relatives is a key topic for future studies in maize.









\item{Asian Rice:}




Introgression appears to have played an important role in the domestication history of rice.
The center of Asian rice (\emph{Oryza sativa}) domestication is not known with complete confidence, but genetic and archaeobotanical evidence point toward both the Yangzee Basin in China and the Ganges plains in India. \mbh{there is a serious debate over whether there were one or two domestication centers of rice...I'd be sure to read up on the most current thoughts about this so that we don't upset potential reviewers}
Domestication occurred 8,200-13,500 BP from the wild species \emph{O. rufipogon} \cite{oka2012origin, fuller2010consilience, ricepedia, molina2011molecular}. \mbh{here limit to one or two citations}
Asian rice readily hybridizes with other domesticated subspecies and with wild relatives (of which there are about twenty \cite{ricepedia}). \mbh{probably best to find a primary literature citation for this (journal article rather than ricepedia).  What other domesticated subspecies are referred to here?  Also, when referring to Asian rice, is this both japonica and indica?  Finally, what are the twenty relatives?  My understanding is gene flow has mostly been with rufipogon and nivara}
The high genetic diversity within \emph{O. sativa} is likely due to introgression from wild relatives both within the domestication center(s) and in new environments where rice has dispersed following domestication \cite{second1982origin}.

Asian rice can be broadly clustered into two groups, japonica and indica \cite{khush2003classifying}.
%The vast majority of the rice varieties fell into the group indica (73.45\%) or japonica (23.02\%).
Japonica rice cultivars were likely domesticated first from wild rice populations in southern China, and indica cultivars later developed through hybridization of ancient japonica with new wild rice populations in south and southeast Asia \cite{Huang2012, londo2006phylogeography}. \mbh{Is this thought to be due to adaptation?  Did the wild rice populations confer local adaptation? If so, I'd state this and explain what type of adaptation this might have been if it's known}
%However, about 1.26\% belonged to four other groups (and 2.27\% were indeterminate).
%Some of the varieties from these other four groups most likely arose from hybrids of cultivated rice and \emph{O. rufipogon}.
Introgression of alleles between cultivated rice populations, as well as between cultivated and wild rice, is common \cite{oka2012origin, second1982origin, zhao2010genomic}\, and natural introgression from wild to domesticated rice is suspected (\cite{zhao2010genomic} calls for research into this possibility). \mbh{strange sentence here...says introgression between cultivated and wild rice is common but then says introgression from wild to cultivated is just suspected...need to clarify}
%These rices often have blight resistance gene, likely imparted from the wild relative parent.
Several resistance genes (grassy stunt virus, bacterial blight, brown planthopper, blast) are known to have been introgressed from wild relatives into \emph{O. sativa} by researchers \cite{brar1997alien, khush1974inheritance}. \mbh{don't think this is what we're after in these case studies.  We want to narrow our focus to natural introgression from wild to crop species during the expansion of the crop. Wild introgression during breeding is interesting, but not the focus of the paper}.
Some aus and rayada varieties retain characteristics of the putative wild parent. \mbh{more background on aus, rayada and aswina...what is there origin? how doe they relate to japonica and indica?}
Aus exhibits a sprawling growth pattern and easy-threshing grain.
Rayada is adapted to areas with longer periods of flooding, with heightened seed dormancy and photoperiod sensitivity (permitting harvest during times without standing water).
Aswina variety, adapted to growth in deepwater conditions with a short period of flooding, seems to be the result of hybridization between \emph{O. sativa} and deepwater-adapted \emph{O. rufipogon} populations. \mbh{need citations regarding these varieties}

In addition to investigative experiments, gene flow from wild relatives has been utilized to produce agronomic rice varieties.
Yatsen No. 1, for example, showed resistance to pests and diseases and adapted well to environmental conditions \cite{ting1933wild}.
Several lines were derived from Yatsen No. 1, and went on to be utilized extensively in parts of China. \mbh{again, don't think this is what we're after with these case studies}





		
	

\item{Barley:}
		
Domesticated and wild barley belong to the same species, \emph{Hordeum vulgare}, and are capable of producing viable offpspring via hybridization \cite{von1995ecographical}.
Barley (\emph{Hordeum vulgare} subsp. \emph{vulgare}) is believed to have been domesticated at least twice roughly 10,000 BP, once from wild subsp. \emph{spontaneum} in the Fertile Crescent and once from subsp. \emph{spontaneum} var. \emph{agriocrithon} in Tibet \cite{takahashi1955origin, badr2000origin, oka2012origin, azhaguvel2007phylogenetic, haberer2015barley}.
However, details of barley domestication are still disputed.


%Variety \emph{agriocrithon} is genetically diverse, and is found throughout much of the range of barley.
%Some have suggested that \emph{agriocrithon} may be the progenitor of six-rowed barley, the product of a hybridization between eastern and western cultivated barley, or a wild-domesticate hybrid \cite{staudt1961origin, zohary1959hordeum, murphy1982origin}\, but \cite{azhaguvel2007phylogenetic}\ dispute these theories, suggesting instead that it arose from ssp. \emph{spontaneum}, perhaps more than once (in Israel as well as in Tibet).
%Presently, the distribution of \emph{spontaneum} consists of the Mediterranean, the Middle-East, and west-central Asia, while other barley wild relatives inhabit Asiatic regions (including Tibet) more broadly \cite{nevo2010drought, harlan1995living, CWR}.
%These wild relatives inhabit regions spanning such abiotic clines as temperature, precipitation, soil type, and altitude, as well as biotic clines \cite{nevo2010drought}.


There has been little genetic investigation into spontaneous barley/\emph{spontaneum} hybrids \cite{ellstrand2003dangerous}.
Barley/\emph{spontaneum} hybrids are fertile, and morphologically intermediate (putatively hybrid) barleys are found when wild and domesticated barleys are grown in sympatry, but hybrids of other wild relatives generally exhibit greatly diminished fertility \cite{ellstrand2003dangerous, harlan1995living}.
Even when the two are not grown immediately adjacent to one another, introgression from wild to domesticate has been shown to happen over distances of more than a kilometer \cite{hillman2001new}.


The barley domestication process has reduced the number of alleles in the domesticate to only 40\% of that found in wild barley, though there remains a great deal of phenotypic diversity among the wild barleys \cite{ellis2000wild}.
The authors of \cite{Poets2015} used STRUCTURE to look for patterns of introgression from wild relatives in a dataset of 803 landraces, and found a high amount variability in the amount of contribution from wild relatives, as well as its location in the genome, within barley populations.
This is indicative of contribution from numerous wild populations.
Furthermore, the authors found that wild introgression contribution is generally greatest from geographically-proximate populations, and that introgressed regions might be combined from geographically-separate wild populations.
Low linkage disequilibrium and small blocks of identity by state indicate that these introgressed regions are old, perhaps dating back to the beginning of barley domestication.
As landraces and nearby wild relatives share similar genomic sequences, the introgressed regions that are exclusive to that landrace are more likely to contain adaptive alleles. 
Such alleles were not identified specifically, though wild-domesticate breeding experiments have shown that wild barleys have alleles for several important agronomic phenotypes, including powdery mildew resistance, brittleness, flowering time, plant height, lodging, and yield \cite{dreiseitl2017heterogeneity,von2006ab,handley1994chromosome}.













\item{Sunflower: }

The common sunflower (\emph{Helianthus annuus}) shows evidence of domestication in eastern United States \cite{harter2004origin, wills2006chloroplast}\, with additional evidence of a possible second origin of domestication in Mexico \cite{lentz2008sunflower}.
Pre-Columbian \emph{H. annuus} distribution spanned much of the Great Plains, from what is now north-central Texas up to and through Montana and North Dakota (see figure 1 of \cite{whitney2010adaptive}).


Domesticated sunflower has long lived in sympatry with wild relatives like \emph{H. petiolaris} and \emph{H. bolanderi} and forms stable hybrid populations \cite{schwarzbach2002likely, rieseberg1988molecular, welch2002patterns}.
Many wild sunflowers are locally-adapted, and weedy hybrid populations share these adaptations \cite{kane2008genetics}.
However, the most striking example of adaptive introgression within \emph{Helianthus} is that of the cucumberleaf sunflower, \emph{H. debilis} ssp. \emph{cucumerifolius}.
Cucumberleaf sunflower is endemic to south-central Texas, and exhibits several adaptations to the region.
Introgressive hybridization imparted locally-adapted alleles from \emph{H. debilis} to \emph{H. annuus} via introgressive hybridization \cite{heiser1951hybridization}. 
These introgressed hybrids formed a new lineage of sunflower (\emph{H. annuus} ssp. \emph{texanus}, \emph{H. a. texanus} hereafter) which displays \emph{H. debilis}-like traits adaptive to south-central Texas climate and ecology.
These adaptive \emph{debilis}-like traits include resistance to herbivorous pests and an increased branching plant architecture, as well as higher overall fitness than \emph{H. annuus} (as measured by higher seed production \cite{whitney2006adaptive}).
Although H. annuus and \emph{H. a. texanus} are interfertile, \emph{H. a. texanus} displays persistent phenotypic differences from \emph{H. annuus} \cite{rieseberg2007hybridization}.


The genome of the common sunflower has been greatly influenced by introgression from wild relatives, due to both natural outcrossing events and concerted breeding efforts in crop improvement.
\emph{Helianthus} has several genes for downy mildew resistance, and each imparts resistance to one or more races of \emph{Plasmopara halstedii}, one of the most agronomically important diseases in sunflower cultivation \cite{cohen1973factors}.
Some of these downy mildew resistence genes were found in wild relatives (including \emph{H. argophyllus}, \emph{H. tuberosus}, and \emph{H. praecox}) and have been successfully bred into modern \emph{H. annuus} \cite{miller1991inheritance}.
PlArg, an allele found in wild silverleaf sunflowers (\emph{H. argophyllus}, inbred line Arg1575-2), confers resistance to all known (20 or more) races of downey mildew \cite{dussle2004pl}\, while others (Pl1-Pl11) are effective for one or more types \cite{rahim2002inheritance}.
Silverleaf sunflower has also been the focus of drought resistance breeding efforts \cite{saucă2010introgression}\ and \emph{Phomopsis} resistance breeding efforts \cite{besnard1997specifying}.
\emph{H. annuus} shows signs of persistent introgressive hybridization with \emph{H. petiolaris} with evidence of positive selection driving some of the genetic differentiation between the two species \cite{yatabe2007rampant}.


Recent investigations into the history of \emph{Helianthus} introgression have implemented genomic methods.
\cite{Baute2015} analyzed transcriptome sequence variation on cultivated and wild \emph{H. annuus}, \emph{H. petiolaris}, and \emph{H. argophyllus}.
Using STRUCTURE, these authors found that introgressions from wild relatives exist on every chromosome in at least one modern line, covering over 10\% of the genome.
Of particular note is the modern line RHA 274, a modern line which was bred with \emph{H. a. texanus} in the 1970s to restore a branching plant body architecture, which allows the plant to produce pollen for a longer period of time, increasing seed production.
RHA 274 has several large introgression from \emph{H. a. texanus}, including one at the site of HaGNAT, the domestication gene associated with branching.
These introgressed regions are not found in the non-branching lines Sunrise and VNIIMK8931, further suggesting that the \emph{H. a. texanus} introgressed regions are causative.




%The species \emph{H. annuus} is a versatile species, capable of adapting to a wide range of environments across the Americas, Europe, Asia, and Australia \cite{kane2008genetics}.
%This versatility may be due in part either to phenotypic plasticity or genetic adaptability \cite{maron2004rapid}.





%whitney2006adaptive
%"these results suggest that introgression of biotic resistance traits was important in the adaptation of H. annuus to central and southern Texas."
%fitness (seed production) high in texanus than in annuus
%identified two pests to which resistance was imparted and important
%Discussion points: 1. texanus has higher fitness than annuus in central texas.
%2. texanus shared traits more in common with debilis than with annuus (herbivore resistance)
%3. 2 of the three traits from point two above are important for adaptation to life in central south Texas.
%The authors ask why these pest-resistance genes have not spread further north beyond this range.  They are uncertain, but conjecture that negative pleiotrophic effects are at play.

%whitney2010adaptive
%"We demonstrate that introgression has altered multiple aspects of the H. annuus phenotype in an adaptive manner, has affected traits relevant to both biotic and abiotic environments, and may have aided expansion of the H. annuus range into central Texas, USA."
%This paper really does show that this case of natural introgression was adaptive.
%The companion paper (2006 Whitney) will likely show the same thing, but I haven't read it yet.

%scascitelli2010genome
%"long-term migration rates were high, genome-wide and asymmetric, with higher migration rates from H. annuus texanus into the two parental taxa than vice versa."
%"H. annuus texanus may serve as a bridge for the transfer of alleles between its parental taxa."
%"contradict recent theory suggesting that introgression should predominantly be in the direction of the colonizing species."

%rieseberg2007hybridization
%this is a very good review.  i think i should use it for the table.
%they also show that traits were imparted from debilis to texanus, and that these are adaptive.  i think they also show genetic evidence, but i'm not sure.














\end{enumerate}

	

	
	
	
	
	
	
	
	




\begin{table}
\rowcolors{2}{white}{gray!25}
\begin{center}
\caption{Programs for identifying genomic regions under introgression} \label{tab:tools}
\begin{tabular}{llll}
\\\toprule  
\rowcolor{white}
{\bf methods}	& {\bf data type } &	{\bf reference} &  {\bf empirical studies } \\ \midrule

\rowcolor{gray!25}
{\emph{\bf chromosome paiting}} &   &   &   \\
\rowcolor{gray!25}
Hapmix	& phased haplotype; reference panel		& Price et al. 2009	&  Hufford et al. 2013;  Suarez et al. 2016 \\ 
\rowcolor{gray!25}
RASPberry &	phased haplotype &	Wegmann et al. 2011	 & Christe et al. 2016 \\
\rowcolor{gray!25}
MultiMix & phased/unphased genotype; reference panel &	Churchhouse and Marchini 2013 &	Eyheramendy et al. 2015 \\
\rowcolor{gray!25}
PCAdmix	 & phased haplotype	 & Brisbin et al. 2012	 & Moreno et al. 2014; Pugach et al. 2016 \\
\rowcolor{gray!25}
LAMP  &	phased haplotypes; reference panel	 & Sankararaman et al. 2008	 & Patterson et al. 2012 \\

\rowcolor{white}
{\emph{\bf phylogenetic relationship}} &   &   &   \\
\rowcolor{white}
ABBA-BABA/D-statistics	 & biallelic SNP  &	Durand et al. 2011	 &  Heliconius Genome Consortium 2012 \\
\rowcolor{white}
fd statistic &	biallelic SNP &	Martin et al. 2015  &	Malinsky et al. 2015; Zhang et al. 2016 \\ 
\rowcolor{white}
five taxon D statistics	& biallelic SNP	&  Pease and Hahn 2015	& Fontaine et al. 2015; Pease et al. 2016 \\

\rowcolor{gray!25}
{\emph{\bf divergence}} &   &   &   \\
\rowcolor{gray!25}
Gmin &	biallelic SNP	&  Geneva et al. 2015	&  Kingan et al. 2015 \\
\rowcolor{gray!25}
RNDmin	& phased haplotype	& Rosenzweig et al. 2016	&  NA \\
\rowcolor{gray!25}
(see .tex file for comment) & biallelic SNP & Racimo et al. 2016 & Sams et al. 2016 \\

%I was not able to add Li's entry to this cell without errors.  I'll put her entry in the line below:
%U_{A,B,C(w,x,y)} and Q95_{A,B,C(w,y)}

\rowcolor{white}
{\emph{\bf population structure related}} &   &   &   \\
\rowcolor{white}
fineStructure &	phased haplotype &	Lawson et al. 2012	& Skoglund et al. 2015 \\
\rowcolor{white}
Globetrotter &	phased haplotype &	Hellenthal et al. 2014 & Scott et al. 2016 \\
\end{tabular}
\end{center}
\end{table} 





\begin{table}
\centering
\begin{adjustbox}{width=1\textwidth}
\small
\label{my-label}
\begin{tabular}{|p{5cm}|p{5cm}|p{2.6cm}|p{2.6cm}|p{2.6cm}|l|}
\hline
Crop & Compatible Wild Relatives & Hybrids and/or Hybridization & Evidence of Crop Introgression & Evidence of Adaptiveness & Source \\ \hline \hline
Maize (\emph{Zea mays} subsp. \emph{mays}) & \emph{Z. m.} subsp. \emph{mexicana}, \emph{Z. m. } subsp. \emph{parviglumis} & X & X & X & \cite{hufford2013genomic} \\ 
\hline 
Asian Rice (\emph{Oryza sativa}) & \emph{O. rufipogon} & X & X & X & \cite{Huang2012} \\ 
\hline
Barley (\emph{Hordeum vulgare}) & \emph{H. v.} subsp. \emph{spontaneum} & X & X & X & \cite{Poets2015} \\ \hline
Sunflower (\emph{Helianthus annuus}) & \emph{H. argophyllus}, \emph{H. bolanderi}, \emph{H. debilis}, \emph{H. petiolaris} & X & X & X & \cite{rieseberg2007hybridization}\\ 
\hline
Cassava (\emph{Manihot esculenta}) & \emph{M. glaziovii} & X & X & X & \cite{bredeson2016sequencing} \\ 
\hline
Potato (\emph{Solanum tuberosum}) & many & X & X & X & \cite{johns1986ongoing} \\
\hline
Tomato (\emph{Solanum lycopersicum}) & \emph{S. pimpinellifolium} & X & X & X & \cite{rick1958role} \\
\hline
Olive (\emph{Olea europaea} ssp. \emph{europaea} var. \emph{sativa}) & \emph{O. e.} ssp. \emph{europaea} var. \emph{sylvestris} & X & X & & \cite{diez2015olive} \\ 
\hline
Soybeans (\emph{Glycine max}) & \emph{G. soja} & X & X &  & \cite{lam2010resequencing} \\ 
\hline
Common Bean (\emph{Phaseolus vulgaris}) & \emph{P. v.} var. \emph{aborigineus, P. v.} var. \emph{mexicanus} [[not in this source]]& X & X &  & \cite{papa2003asymmetry} \\
\hline
Grapes (\emph{Vitis vinifera} subsp. \emph{vinifera}) & \emph{V. v.} subsp. \emph{sylvestris} & X & X &  &  \cite{myles2011genetic} \\
\hline
Sorghum (\emph{Sorghum bicolor} subsp. \emph{bicolor}) & \emph{S. b.} subsp. \emph{arundinaceum, S. b.} subsp. {drummondii} & X & X &  & \cite{aldrich1992patterns} \\
\hline
Wheat (\emph{Tritium monococcum, T. dicoccum, T. aestivum}) & \emph{T. m. boeoticum, T. dioccoides, T. urartu, Aegilops speltoides, A. tauschii} & X & X &  & \cite{zohary1969wild} \\
\hline
Apple (\emph{Malus domesticus}) & \emph{M. sylvestris}, \emph{M. orientalis}, \emph{M. baccata}, \emph{M. sieversii}  & X & X & & \cite{cornille2012new} \\
\hline
\end{tabular}
\end{adjustbox}
\end{table}

\begin{figure}[h]
	\centering
	\includegraphics[width=17.35cm]{par_figure.png}
	\caption{Map of the natural ranges of wild relatives of four domesticated crops, overlayed with average annual temperature.}
	\label{boxplot:map}
\end{figure}

\begin{figure}[h]
	\centering
	\includegraphics[width=17.35cm]{boxplot.png}
	\caption{The distribution of average annual temperature experienced in the geographic home ranges of wild relatives interfertile with four crops}
	\label{fig:map}
\end{figure}

\begin{figure}[h]
	\centering
	\includegraphics[width=17.35cm]{li_figure.png}
	\caption{Li's caption here.}
	\label{fig:map}
\end{figure}























\section*{Re-evaluating concepts of domestication}

A framework in which crops were domesticated from a single population or even a single species is, in several instances, an oversimplification.
An history of introgression during diffusion appears to be the rule for crops rather than the exception.
Theory suggests that colonizing species will overwhelmingly be recipients of introgression from locally-adapted native species \cite{Currat2008}.
Crops, given their frequent history of diffusion from defined centers of origin, are therefore potential recipients of adaptive introgression.



With this in mind, certain aspects of crop evolution must be re-evaluated:

* Estimates of the initial domestication bottleneck may be skewed when introgression is not considered.
Chromosomal regions experiencing introgression may have an altered effective population size ($N_e$) relative to non-introgressed regions depending on diversity within the donor taxon.
For example, introgression from wild taxa with historically high $N_e$ will lead to underestimates of the strength of the domestication bottleneck.
Conversely, if the donor population has a relatively small effective population size, the opposite bias may be imposed upon bottleneck estimations.
In most cases, the effective population size of the domesticated crop will be lower than that of the wild progenitor and wild relative populations.

* Estimates of the timing of domestication based on levels of sequence divergence may be affected when introgressed haplotypes are included.
The directionality of this effect is likewise dependent on $N_e$ of the donor population.


* Loci under selection during domestication are often identified based on signatures of substantially-reduced nucleotide diversity in the domesticated taxon relative to the wild progenitor and high allele frequency differentiation between these taxa.
Introgression may alter these signatures and confound detection of domestication loci.

%\lwang{I donot quite understand this point. So, it is assumed that the introgressed haplotype has higher diversity, which renders difficulty to detect selection in such regions, right? But the case in maize is that even the introgressed haplotypes had higher diversity, but as selection has act long time on the region, the observed current diversity has already been reduced.}


* When  highly-introgressed or hybrid populations are selected for domestication (as in the case of potato and tomato), identification of original progenitor(s) and domestication centers is difficult.
Determining whether a crop 
Crops arising from highly-interfertile compleces of wild relatives 

%\gmj{It may not fit here perfectly, but we might consider a point about how crop-wild introgressions can also mask or obscure even the true progenitor species and center(s) of domestication.  The examples of potato and tomato domestication comes to mind; in potato, persistent and thorough interbreeding with a complex of wild relatives during domestication probably make it impossible to identify a single progenitor (if ever there was one), and in tomato, widescale recent introgression between two or three wild relatives confound attempts to identify which is the progenitor, and therefore the center of domestication as well.}










\section*{Future studies in crop-wild introgression}

Research has so far shown that adaptive crop-wild introgression has played a significant role in the domestication histories of many agronomically-important crops.
However, the dynamics of the process in these cases are not yet fully understood.
To what extent does the level of introgression across taxa depend on divergence time and/or mutation load between donor and recipient taxa?
Can colonizing species and/or hybrid swarms serve as bridges for gene flow between previously allopatric taxa?
At what geographic scale does adaptive introgression occur?
Is introgression frequently restricted to very local populations, or is it often seen over broad geographic ranges?
To what extent does this depend on the slope of environmental gradients such as temperature, precipitation, and elevation?
How did the conscious, subconscious, and unconscious decisions of early farmers facilitate or hinder adaptive introgression into their crops during early domestication?
How do the practices of contemporary farmers affect the process of adaptive introgression today?

Additional study of introgression in agroecosystems could lead to advances in both basic and applied genetics, and specifically the continued improvement of modern crops.
Loci underlying the domesticated phenotype can be more clearly identified by removing the confounding population genetic signal of introgression.
These loci are potentially beneficial targets for crop improvement.
Furthermore, adaptive introgression that is clearly tied to a specific environment may include beneficial alleles that can be utilized in crop breeding.








\section*{Conclusions}

The study of crop domestication has been revolutionized by the advent and application of genomic tools.
The genomes of crops and their wild relatives tell a story of give-and-take that extends well beyond the initial stages of domestication.
Likewise, population genetic theory reinforces the proclivity of wild relatives to provide advantageous, locally-adapted alleles to crops as they disperse beyond their domestication centers into new geographies with new ecological pressures and niches.











\bibliography{bib_gj}

\end{document}


\end{document}



