\documentclass[11pt]{article}
\usepackage{color}
\usepackage{authblk}%allows footnote format for authors
\usepackage[letterpaper, margin=1in]{geometry} %package that allows changes in margins and header/footers
\usepackage[numbers,sort]{natbib}
\bibliographystyle{ieeetr}
\newcommand{\mbh}[1]{\textcolor{blue}{ \emph{\scriptsize  #1}} } %creating command for Matt's comments
\newcommand{\lwang}[1]{\textcolor{red}{ \emph{\scriptsize  #1}} } %creating command for Li's comments
\newcommand{\gmg}[1]{\textcolor{green}{ \emph{\scriptsize  #1}} } %creating command for Garrett's comments

\title{Review: Crop adaptation through post-domestication introgression from wild relatives}

\author[1]{Authors: Garrett M. Janzen}%author information
\author[1]{Li Wang}
\author[1,*]{Matthew B. Hufford}
\affil[1]{Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa, USA}
\affil[*]{Correspondence: mhufford@iastate.edu (M.B. Hufford)}
\date{}

\begin{document}

\maketitle

The traditional paradigm in crop domestication has been origin from a wild relative within one or more defined geographic centers followed by expansion to the modern-day extent of cultivation.
Absent from this paradigm are the effects of hybridization between diffusing domesticates and closely-related, locally-adapted wild relatives outside the center of origin.
New methods have recently been employed to detect genome-wide patterns of introgression in a number of species.
In this review, we will: 1) briefly describe these methods and provide a summary of their recent application for detection of crop-wild introgression, 2) review evidence supporting the hypothesis that wild-to-crop introgression has conferred local adaptation, 3) consider how the prevalence of this introgression alters traditional concepts of domestication, and 4) describe future advances in both basic and applied genetics that can be made through the study of introgression in agroecosystems.

\section*{Introgression methods and their recent applications}
The recent availability of genome-wide resequencing and reduced-representation genotyping (\emph{e.g.}, GBS and RAD-Seq) data combined with new analytical methods have facilitated comprehensive study of introgression across a number of species (\textbf{Table 1}).
\begin{enumerate}
\item{High-density marker data can be used with haplotype-based and other methods to assign specific genomic regions to a taxon of origin and to identify introgression across taxa.}
\item{In several instances, introgression has been found to be conserved across individuals and populations suggesting an adaptive role:}
	\begin{enumerate}
	\item{Adaptive introgression has been discovered from Neanderthals and other archaic hominins into humans at loci controlling skin pigmentation, defense against pathogens, and tolerance of high altitude (reviewed in \cite{Racimo2015}}).
	\item{Introgression across butterfly species at protective color-pattern loci has conferred M\"{u}llerian mimicry (\cite{Heliconius2012}; \textbf{Figure 1})}.
	\item{Adaptive introgression across mosquito species has spread insecticide resistance \cite{Norris2015}}.
	\item{Introgression across \emph{Mimulus} (\emph{i.e.}, monkeyflower) species has resulted in adaptation to pollinator preference and has contributed to speciation \cite{Stankowski2015}}.
	\end{enumerate}
\item{Identification of introgression across a wide range of taxa at loci controlling a variety of adaptive traits suggests it has been an important evolutionary force.}
\end{enumerate}

\noindent{\textbf{Table 1:} List and brief description of recently developed methods and examples of empirical studies employing these methods.}

\noindent{\textbf{Figure 1}: Wing coloration patterns in \emph{Heliconius} and evidence for introgression across species based on Patterson's \emph{D}-statistic; adapted from \cite{Heliconius2012}.}

\section*{Crop adaptation through introgression}
Over the last few years, several high-profile publications based on genome-wide data have documented introgression between crops and wild relatives outside putative domestication centers. A history of introgression during diffusion appears to be rule for crops rather an exception.
\begin{enumerate}
	\item{Crops, given their frequent history of diffusion from defined centers of origin, are ideal recipients of adaptive introgression. Theory suggests that such colonizing species will overwhelmingly be recipients rather than donors of introgression \cite{Currat2008}.}
	\item{Empirical studies are increasingly revealing that introgression has occurred in many of the world's most important crops (\textbf{Table 2}).}
	\begin{enumerate}
		\item{Maize: As maize spread from a lowland center of origin it received introgression from a highland-adapted wild relative \cite{Hufford2013}.}
		\item{Sunflower: Domesticated sunflower has received substantial introgression from wild relatives post-domestication that has potentially reintroduced a branching morphology \cite{Baute2015}.}
		\item{Rice: \cite{Huang2012}}
		\item{Barley: Landraces of barley show shared ancestry with geographically-proximate populations of wild relatives suggesting introgression \cite{Poets2015}.}
		\item{Olive: \cite{Diez2015}}
		\item{Cassava: \cite{Bredeson2016}}
	\end{enumerate}
	\item{Predictions regarding the likelihood of adaptive introgression in additional crops can be made through comparison of their centers of origin to their current extent of cultivation and distribution of wild relatives across environmental gradients (\textbf{Figure 2}).}
\end{enumerate}

\noindent{\textbf{Table 2:} Overview of literature evaluating potential for crop-wild introgression in the world's 15 most important crops including a column of references and a column summarizing major findings.}

\noindent{\textbf{Figure 2}: Multi-panel figure showing maps of crop centers of domestication, distributions of wild relatives, and current extent of cultivation across gradients of (a) temperature, (b) precipitation, and (c) elevation.}

\section*{Reevaluating concepts of domestication}

\section*{Future studies in crop-wild introgression}

\bibliography{TiG_references}

\end{document}